\documentclass[11pt]{article}
\usepackage[margin=1in]{geometry}
\usepackage{lineno} % for line numbers
\usepackage{graphicx}
\usepackage{bm} % for bold math characters
\usepackage{booktabs} % for prettier hlines in table
\usepackage{subcaption} % for subfigures
\usepackage[affil-it]{authblk} % for footnote-style author affiliation
\usepackage{setspace} % for 1.5-spaced lines not including footnotes, floats
\usepackage{tikz} % for graphical model drawing
\usepackage{float} % to enforce figure positioning in subsection they're declared in
\usepackage{amsmath}
\usepackage[singlelinecheck=false]{caption} % to left-justify single-line table captions
\usepackage{array}
\newcolumntype{P}[1]{>{\centering\arraybackslash}p{#1}} % for centered, width-specified columns 
\usepackage{makecell} % for multi-row table entries
\usepackage{tabularx} % for tables that don't spill past margin
\usepackage[markup=underlined]{changes}

\usepackage{url} % otherwise bibtex hates my references file
\usepackage[round,sort]{natbib}
\linespread{1.3}

% Tikz preamble from Michael Landis
\usepackage{amssymb} %maths
\usepackage{amsmath} %maths
%\usepackage{subfig} % can't be used together with subcaption package
\usepackage{skull}
\usetikzlibrary{positioning}
\usetikzlibrary{arrows}
\usetikzlibrary{fit}
\usetikzlibrary{calc}
\usetikzlibrary{automata}
\usetikzlibrary{decorations.markings}
\usetikzlibrary{decorations.pathreplacing} % for curly bracehttps://www.overleaf.com/project/6127a9ffb8327d4ab8b9fd4d
\usetikzlibrary{shapes.multipart}
\tikzset{>=latex}
\tikzstyle{snode}=[black,draw=black,line width=1.5pt,shape=circle,fill=white,minimum size=8mm]
\tikzstyle{obnode}=[black,draw=black,line width=1.5pt,shape=circle,fill=black!20!white,minimum size=8mm]
\tikzstyle{detnode}=[black,draw=black,line width=1.5pt,densely dotted,shape=circle,fill=white,minimum size=8mm]
\tikzstyle{constnode}=[black,draw=black,line width=1.5pt,shape=rectangle,fill=white,minimum size=8mm]
\tikzstyle{blnode}=[white,draw=black,line width=1pt,shape=circle,fill=black,minimum size=1mm,font=\scriptsize,inner sep=1pt]
\tikzstyle{ylnode}=[black,draw=black,line width=1pt,shape=circle,fill=yellow,minimum size=1mm,font=\scriptsize,inner sep=1pt]
\tikzstyle{taro}=[->,line width=2pt,color=black]
\tikzstyle{baro}=[<->,line width=2pt,color=black]
\tikzstyle{bline}=[line width=2pt,color=black]
\tikzstyle{dtaro}=[->,line width=2pt, densely dotted,color=black]
\tikzstyle{dline}=[line width=2pt, densely dotted,color=black]
\tikzstyle{smod}=[black, draw=black, line width=2pt, fill=white, shape=rectangle, rounded corners, minimum size=10mm, minimum width=20mm]
\tikzstyle{obmod}=[black, draw=black, line width=2pt, fill=black!20!white, shape=rectangle, rounded corners, minimum size=10mm, minimum width=20mm, minimum width=20mm]

% For proof
\usepackage{amsthm}
\newtheorem{theorem}{Theorem}

% Set up supplemental figure numbering
\newcommand{\beginsupplement}{\setcounter{figure}{0}
        \renewcommand{\thefigure}{S\arabic{figure}}
        \setcounter{table}{0}
        \renewcommand{\thetable}{S\arabic{table}}}

\begin{document}

% \begin{flushright} % for MBE submission
% Submission type: Article\\
% Section: Methods
% \end{flushright}

\begin{center}
	\Large{\textbf{A phylogeny-aware GWAS framework to correct for heritable pathogen effects on infectious disease traits}}
\end{center}
\vspace{4pt}

\begin{flushleft}
	Sarah Nadeau$^{1,2}$, Christian W. Thorball$^{3}$, Roger Kouyos$^{4,5}$, Huldrych F. Günthard$^{4,5}$, Jürg Böni$^{4}$, Sabine Yerly$^6$, Matthieu Perreau$^{7}$, Thomas Klimkait$^{8}$, Andri Rauch$^{9}$, Hans H. Hirsch$^{8,10,11}$, Matthias Cavassini$^{12}$, Pietro Vernazza$^{13}$, Enos Bernasconi$^{14}$, Jacques Fellay$^{2,3,15}$, Venelin Mitov$^{\dagger,1,2}$, Tanja Stadler$^{\dagger,\ast,1,2}$, and the Swiss HIV Cohort Study (SHCS)
\end{flushleft}

\noindent
$^{1}$Department of Biosystems Science and Engineering, ETH Zurich, Basel, Switzerland\\
$^{2}$Swiss Institute of Bioinformatics, Lausanne, Switzerland\\ 
$^{3}$Precision Medicine Unit, Lausanne University Hospital and University of Lausanne, Lausanne, Switzerland\\ 
$^{4}$Institute of Medical Virology, University of Zurich, Zurich, Switzerland\\ 
$^{5}$Division of Infectious Diseases and Hospital Epidemiology, University Hospital Zurich, University of Zurich, Zurich, Switzerland\\
$^{6}$Division of Infectious Diseases, Laboratory of Virology, Geneva University Hospital, Geneva, Switzerland\\
$^{7}$Division of Immunology and Allergy, University Hospital Lausanne, Lausanne, Switzerland\\
$^{8}$Department of Biomedicine, University of Basel, Basel, Switzerland\\
$^{9}$Department of Infectious Diseases, Bern University Hospital and University of Bern, Bern, Switzerland\\
$^{10}$Division of Clinical Virology, University Hospital Basel, Basel, Switzerland\\
$^{11}$Division of Infectious Diseases and Hospital Epidemiology, University Hospital Basel, Basel, Switzerland\\
$^{12}$Division of Infectious Diseases, University Hospital Lausanne, Lausanne, Switzerland\\
$^{13}$Division of Infectious Diseases, Cantonal Hospital St. Gallen, St. Gallen, Switzerland\\
$^{14}$Division of Infectious Diseases, Regional Hospital Lugano, Lugano, Switzerland\\
$^{15}$Global Health Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne, Lausanne, Switzerland\\
$\dagger$Co-last authors\\
$*$Corresponding author: tanja.stadler@bsse.ethz.ch

\newpage
\begin{linenumbers}
% \begin{doublespace}

\begin{abstract}
	Infectious diseases are particularly challenging for genome-wide association studies (GWAS) because genetic effects from two organisms (pathogen and host) can influence a trait. Traditional GWAS assume individual samples are independent observations. However, pathogen effects on a trait can be heritable from donor to recipient in transmission chains. Thus, residuals in GWAS association tests for host genetic effects may not be independent due to shared pathogen ancestry. We propose a new method to estimate and remove heritable pathogen effects on a trait based on the pathogen phylogeny prior to host GWAS, thus restoring independence of samples. In simulations, we show this pre-processing step can increase GWAS power to detect truly associated host variants when pathogen effects are highly heritable, with strong phylogenetic correlations. We applied our framework to data from two different host-pathogen systems. For HIV-1 set-point viral load (spVL) in the Swiss HIV Cohort Study, we recover slightly weaker but qualitatively similar signals of association between spVL and human genetic variants in the \emph{CCR5} and major histocompatibility complex (MHC) gene regions. Our simulation study confirms that based on the estimated heritability and selection parameters for HIV-1 spVL, standard GWAS are robust to pathogen effects. For \emph{A. thaliana} quantitative disease resistance (QDR) to \emph{X. arboricola}, phylogenetic correlations are not strong enough to justify our phylogenetic correction for GWAS. In summary, our framework provides additional information on the evolutionary dynamics of traits in pathogen populations and may improve GWAS if pathogen effects are highly phylogenetically correlated amongst individuals in a cohort.
\end{abstract}

\section*{Introduction}

% What is GWAS and why is it important
A key goal of genome-wide association studies (GWAS) is to understand the genetic basis of phenotypic variation among individuals. In a typical GWAS, millions of genetic variants from across an organism's genome are screened for statistical association with a trait of interest. Ideally, this procedure identifies variants that are located in, or are in linkage disequilibrium with, alleles that directly affect the trait. If GWAS finds a variant strongly associated with a disease trait, the gene product may be a good drug target \citep{Okada2014}. Even if no single variant has a strong association, many small associations can be aggregated into a polygenic risk score to identify susceptible individuals \citep{Dudbridge2013}.

% Overview of relevant methods and why we don't use them
It is well-known that GWAS can be sensitive to confounding variables. Shared ancestry among individuals, especially between close relatives, can give rise to spurious genetic correlations with a trait. Corrections for these types of population structure in human GWAS cohorts are well-developed and widely accepted  \citep{Astle2009, Price2006}. More recently, analogous methods have been developed for microbial GWAS, where clonal reproduction exacerbates population structure \citep{Power2017}. Microbial GWAS-specific phylogenetic methods to account for population structure in microbial GWAS include explicitly testing for lineage-specific effects as in \citep{Earle2016} and modified association tests that account for phylogenetic relationships amongst samples as in \citep{collins_phylogenetic_nodate}. 
% \citep{Earle2016} simultaneously test for pathogen lineage and locus-level associations with a trait using a linear mixed model framework. \citep{collins_phylogenetic_nodate} test for locus-level associations that are significantly stronger than expected under a null model considering shared pathogen ancestry. 
These approaches are designed to quantify genetic effects from one organism on a trait. 

In the infectious disease context, genetic effects from two organisms - the host and the pathogen - may affect an infectious disease trait. GWAS using paired host-pathogen genotype data have previously been done to elucidate the marginal and interaction effects of host and pathogen genetic variants. Methods to account for microbial population structure when testing for marginal host associations or host-pathogen interaction effects include adding the microbial kinship matrix as a random effect in a linear mixed model as in \citep{Wang2018Two-wayGenomes} and using principle components derived from either this matrix or the pathogen phylogeny as covariates in a linear model as in \citep{Naret2018}. These methods focus on capturing and accounting for correlations due to the pathogen phylogeny, without further investigating the nature of these correlations.
%However, the top principle components capture only some of the information from the full pathogen phylogeny. Furthermore, we would like to simultaneously address issues (i) and (ii).

In this work, we draw from the field of phylogenetic comparative methods to propose a new two-step framework to correct for pathogen population structure and satisfy the GWAS assumption of independent samples. It is a pre-processing step for GWAS that first fits an evolutionary model to trait data and the pathogen phylogeny to estimate the correlation structure of the trait due to pathogen effects. On the basis of this inference, it estimates and removes pathogen effects on the trait so that downstream GWAS association testing with host genetic variants can be performed as normal under the assumption of independent samples. The main advantage of this approach compared to the previously outlined methods to correct for pathogen population structure are that it generates additional information on the evolutionary dynamics of the trait in the pathogen population. It is also restricted to a pre-processing step, which allows use of existing highly optimized tools to perform GWAS association tests under a variety of models. This framework relies on paired pathogen-host genotyping and is envisioned specifically for continuous-valued traits that are highly heritable from infection partner to infection partner.
% We hypothesis this should increase GWAS power for several reasons. For one, pathogen effects may be confounding or extraneous variables that produce distorted or spurious associations between host genetic variants and a disease trait in host GWAS. 
We hypothesized that our approach should improve GWAS power to identify host genetic variants broadly associated with disease traits.
%This may occur if the virus affects the disease trait (the dependent variable in host GWAS) or if it affects both the disease trait and the host genotype (the independent variable in host GWAS).
%For example, stratification in both the host and pathogen populations may yield spurious correlations in host GWAS that are really due to independent pathogen factors. 
% Then, co-evolution between a pathogen and its host may yield distorted estimates for host genetic effects that are not broadly protective against all pathogen strains. Finally, trait values from a population are not truly independent samples, given that hosts closer in the transmission chain have more similar pathogen strains and therefore more similar trait values. 

% % Why is spVL and important trait
% For HIV, GWAS have used a trait called set point viral load (spVL) to identify human variants associated with the severity of disease course. spVL is generally defined to be the average concentration of viral RNA copies in host plasma during the asymptomatic phase of infection in the absence of treatment (see e.g. \cite{Alizon2010}). In untreated individuals, spVL is predictive of the duration of asymptomatic infection \citep{Mellors1996} and infectiousness \citep{Quinn2000}. If viral load can be reduced to undetectable levels, an individual is effectively uninfectious and the risk of disease progression is massively reduced \citep{HHSARTGuidelines2019}. Notably, spVL varies by orders of magnitude between individuals \citep{Mellors1996}. Thus, spVL measurements point to a wide range in natural HIV control amongst individuals. 

% % How do we measure the extent to which genetic factors determine spVL
% To understand HIV pathogenicity, it is important to understand to what extent spVL is determined by host genetic factors as opposed to pathogen genetic factors. Heritability is a key measure of how genetically-determined a trait is. On the host side, narrow-sense heritability, the fraction of total trait variance explained by additive genetic effects, is a typical output of GWAS. 
% % How heritable is spVL (human side)
% Several GWAS have measured the narrow-sense heritability of spVL and identified associated host genetic variants \citep{Bartha2013, Dalmasso2008, Fellay2007, Pereyra2010, Fellay2009, Pelak2010, VanManen2011, McLaren2012}. The largest study to-date by \citet{McLaren2015} estimated that additive human genetic effects account for approximately 25\% of the variation in spVL amongst infected individuals. All but 5\% of this was attributed to two regions in the human genome, the major histocompatibility complex (MHC) and C-C motif chemokine receptor 5 (\emph{CCR5}). Both associations are biologically relevant: the MHC encodes proteins that present viral epitopes at the cell surface and \emph{CCR5} encodes a co-receptor for HIV-1 cell entry. In other words, MHC proteins match bits of the virus like puzzle pieces and display these to signal that a cell is infected. CCR5 proteins help the virus infect target cells.

% How heritable is spVL (viral side)
% On the pathogen side, broad-sense heritability $H^2$ is typically measured. This is the fraction of total trait variance due to any factors ``inherited'' from infection partner to infection partner (the most obvious being pathogen genetic factors). Broad-sense heritability is commonly measured by quantifying how correlated a trait is between infection partners or how phylogenetically structured a trait is based on a reconstructed pathogen phylogeny. For HIV spVL, estimates differ depending on the methods employed and the cohort studied (see \citet{Mitov2018} for a discussion of this uncertainty). Using phylogenetic methods on large UK and Swiss cohorts, \citet{Mitov2018} and \citet{Bertels2018} estimated that pathogen factors account for 21\% - 29\% of the variation in spVL amongst infected individuals. However, variation in the MHC is known to exert strong selective pressure on the virus \citep{Kloverpris2016, Nguyen2021SystematicLink}. If the virus can change its ``puzzle piece'' shape to escape MHC-presentation, infected cells can go undetected. This means that MHC variants affect spVL largely via selection on the virus \citep{Bartha2017}. In summary, human genetic factors play a role in determining spVL, but these effects may be due to interaction with specific viral genetic variants.

% % The gap
% Most of the GWAS for human genetic determinants of spVL conducted so far \citep{Dalmasso2008, Fellay2007, Pereyra2010, Fellay2009, Pelak2010, VanManen2011, McLaren2012, McLaren2015} do not explicitly consider any viral effect on spVL. In these GWAS, viral genetic effects are lumped in with residual variance due to other, non-genetic factors. This has several potential negative consequences. 
% n GWAS, hosts are typically selected to not share recent ancestry. Further, shared environmental variables are included as covariates to avoid spurious associations. We argue similar considerations must be taken
% for pathogen effects that heritable from one transmission partner to another \citep{Leventhal2016}. 

% Introduction to our method
In the following, we describe the evolutionary model for heritable, continuous-valued infectious disease traits upon which our method is based. We then derive a maximum likelihood estimate for the pathogen part of a trait under this model, and describe a new infectious disease GWAS framework using this information. In simulations, we show that this framework can improve GWAS power to detect host genetic variants that affect disease traits. Finally, we apply our framework to paired host-pathogen genotyping data from the Swiss HIV Cohort Study (SHCS) and a previously studied \emph{Arabidosis thaliana}-\emph{Xanthomonas arboricola} pathosystem. We show that associations with set-point viral load (spVL) and quantitative disease resistance (QDR) traits, respectively, are robust to a correction for pathogen effects.

\section*{New Approaches}
\subsection*{A statistical model for heritable, continuous-valued infectious disease traits}

Variation in infectious disease traits like viral load or infection severity can come from several sources. These include host genetic factors, pathogen genetic factors, interaction effects between the host and the pathogen, or non-genetic factors like healthcare quality or temperature. GWAS typically stratify samples or include covariates to correct for host genetic factors or non-genetic factors that may be correlated with a trait value. This leaves pathogen genetic factors as a remaining source of correlation, since close transmission partners may be infected with very similar pathogen strains. Broad-sense pathogen heritability $H^2$ quantifies the fraction of total variance in a trait that is ``inherited'' from infection partner to infection partner, i.e., due to pathogen factors. To characterize $H^2$ and the heritable and non-heritable factors that determine infectious disease traits, we use a phylogenetic mixed model (PMM) \citep{Housworth2004}. PMMs assume continuous traits are the sum of independent heritable and non-heritable parts. In the infectious disease GWAS case, we assume the heritable part comprises pathogen genetic factors and all other factors are non-heritable. The heritable pathogen part is modeled by a random process occurring in continuous time along the branches of the pathogen phylogeny, as in Figure \ref{fig:spVLModel}A. The non-heritable part is modeled as Gaussian noise added to sampled individuals at the tips of the phylogeny.

\begin{figure}[H]
	\begin{center}
	\includegraphics[width = \linewidth]{figures/simulation_setup.png}
	\caption{A high-level schematic of our POUMM-based simulation framework in the context of HIV-1 spVL. (A) shows how the viral effects on spVL evolve along the viral phylogeny according to an Ornstein-Uhlenbeck process. (B) shows how human host genetic effects are the sum of independent effects from several causal variants. Each variant can be present in 0, 1, or 2 copies. Half the variants have a positive effect of size $\delta$ and half have a negative effect of size $\delta$. (C) shows how other environmental effects are independently drawn from a Gaussian distribution centered at 0. These three effects sum to the trait value for each simulated individual.}
		\label{fig:spVLModel}
	\end{center}
\end{figure}

PMMs have previously been applied to the study of infectious disease traits using two different types of random processes to model trait evolution. The Brownian Motion (BM) process assumes unbounded trait values, i.e. the trait can attain any value. The Ornstein-Uhlenbeck (OU) process assumes trait values fluctuate around an optimal value, i.e. extreme trait values are unlikely. Here, we assume the more flexible OU process as it encompasses a wider variety of evolutionary scenarios. For example, \citet{Mitov2018} and \citet{Bertels2018} previously showed the OU process has higher statistical support for HIV-1 spVL. This makes sense given that spVL is likely under stabilizing selection to maximize viral transmission potential \citep{Fraser2014}. The full model is called the phylogenetic Ornstein-Uhlenbeck mixed model (POUMM) and is described in detail by \citet{Mitov2018}. Here, we review the main points relevant to our method.

Under the POUMM, the trait $z$ is the sum of heritable genetic effects $g$, i.e. due to the pathogen, and non-heritable ``environmental'' effects $\epsilon$, i.e. host genetic effects and other environmental or interaction effects: 

\begin{equation}
    z = g + \epsilon
\end{equation}
$g$ is a pathogen trait that evolves along the phylogeny according to an OU process. The OU process is defined by a stochastic differential equation with two terms. The first term represents a deterministic pull towards an optimal trait value and the second term represents stochastic fluctuations modelled by Brownian motion \citep{Butler2004}:

\begin{align}
\begin{split}
	&dg(t) = \alpha[\theta - g(t)]dt + \sigma dW_t \\
	&g(0) = g_0 
	\label{eq:OUprocess}
\end{split}
\end{align}

Here the parameter $\alpha$ represents selection strength towards an evolutionarily optimal value represented by parameter $\theta$. The parameter $\sigma$ measures the intensity of stochastic fluctuations in the evolutionary process. Finally, $dW_t$ is the Wiener process underlying Brownian motion. The OU process is a Gaussian process, meaning that $g(t)$ is a Gaussian random variable. Assuming $g(t)$ starts at initial value $g_0$ at time $t = 0$ at the root of the phylogeny, we can write the expectation for $g(t)$ at time $t$: 

\begin{equation}
   E[g(t)] = g_0e^{-\alpha t} + (1 - e^{-\alpha t})\theta \label{eq:OUmean}
\end{equation}
and the variance in $g(t)$ if we were to repeat the random evolutionary process many times \citep{Butler2004}:
\begin{equation}
    Var[g(t)] = \frac{\sigma^2}{2\alpha}(1 - e^{-2\alpha t}) \label{eq:OUVar}
\end{equation}

$g$ evolves independently in descendent lineages after a divergence event in the phylogeny. The covariance between $g(t)$ in a lineage $i$ at time $t_i$ and another lineage $j$ at time $t_j$, $Cov\big(g_{i}(t_i), g_{j}(t_j)\big)$, increases with the amount of time between $t_0$ and the divergence of the two lineages, $t_{0(ij)}$, and decreases with the total amount of time the lineages evolve independently, $d_{ij}$ \citep{Butler2004}: 

\begin{equation}
	Cov\big(g_{i}(t_i), g_{j}(t_j)\big) = \frac{\sigma^2}{2\alpha}[e^{-\alpha d_{ij}}(1 - e^{-2\alpha t_{0(ij)}})]
	\label{eq:OUcov}
\end{equation}
Next, we remember that $\epsilon$ is the non-heritable part of the trait. $\epsilon$ is modeled as a Gaussian random variable that is time- and phylogeny-independent. The expectation of $\epsilon$ is 0, meaning environmental effects are equally likely to raise or lower the trait from the pathogen-determined level. The parameter $\sigma_\epsilon^2$ measures the between-host variance of the environmental effect.

\begin{align}
\begin{split}
	E(\epsilon) = 0 \\
	Var(\epsilon) = \sigma^2_\epsilon
\end{split}
\end{align}

Finally, broad-sense trait heritability can be calculated as the fraction of total trait variance that is heritable:

\begin{equation}
	H^2_{t} = \frac{Var[g(t)]}{Var[g(t)] + Var(\epsilon)} = \frac{\frac{\sigma^2}{2\alpha}(1 - e^{-2\alpha t})}{\frac{\sigma^2}{2\alpha}(1 - e^{-2\alpha t}) + \sigma^2_\epsilon}
	\label{eq:POUMM-H2}
\end{equation}

\subsection*{Teasing apart pathogen and non-pathogen effects on a trait}

Given the assumptions of the POUMM, we can estimate a heritable pathogen effect on a trait and a corresponding non-heritable, host and environmental effect. Here, we derive a maximum-likelihood estimate for these values for individuals in a GWAS cohort, given measured trait values and a pathogen phylogeny linking the infecting strains.

Let $\bm{g}(\bm{t})$ be a vector of $g$ values, one for each individual in the cohort. $\bm{t}$ are the sampling times of each individual relative to the root of the phylogeny. To simplify notation, we omit the $\bm{t}$ from here on. $\bm{g}$ is a realization of a Gaussian random vector $\bm{G} \sim \mathcal{N}\big(\bm{\mu}_{OU}, \boldsymbol{\Sigma}_{OU}\big)$. The expectation $\bm{\mu}_{OU}$ is defined by equation \ref{eq:OUmean}, the diagonal elements of the covariance matrix $\boldsymbol{\Sigma}_{OU}$ are defined by equation \ref{eq:OUVar}, and the off-diagonal elements of $\boldsymbol{\Sigma}_{OU}$ by equation \ref{eq:OUcov}. Similarly, let $\bm{\epsilon}$ be a vector of the environmental part of the trait for each individual.  $\bm{\epsilon}$ is a realization of a Gaussian random vector $\bm{\mathcal{E}} \sim \mathcal{N}\big(\bm{0}, \boldsymbol{\Sigma}_\mathcal{E}\big)$, where $\boldsymbol{\Sigma}_\mathcal{E}$ is a diagonal matrix with diagonal elements equal to $\sigma^2_\epsilon$.

Considering that $\bm{G}$ and $\bm{\mathcal{E}}$ are independent random vectors and that their realizations $\bm{g}$ and $\bm{\epsilon}$ must sum together to equal the observed trait values $\bm{z}$, we can write the following proportionality for the joint probability density of $\bm{g}$ and $\bm{\epsilon}$:

\begin{equation}
	f\big(\bm{g}, \bm{\epsilon}\big) \propto \mathcal{N}\big(\bm{g}; \bm{\mu}_{G}, \boldsymbol{\Sigma}_G\big)
	\label{eq:pdfGprop}
\end{equation} 

where the expected value of $\bm{g}$ and the covariance matrix $\boldsymbol{\Sigma}_G$ are defined as:

\begin{align}
	Exp(\bm{g}) = \bm{\mu}_{G} &=  \boldsymbol{\Sigma}_G\big(\boldsymbol{\Sigma}_{OU}^{-1}\bm{\mu}_{OU} + \boldsymbol{\Sigma}_\mathcal{E}^{-1} \bm{z}\big) \label{eq:MuG}\\
	\boldsymbol{\Sigma}_G &= \big(\boldsymbol{\Sigma}_{OU}^{-1} + \boldsymbol{\Sigma}_\mathcal{E}^{-1}\big)^{-1} \label{eq:SigmaG}
\end{align}

\begin{proof}
	\begin{align}\label{eq:pdfG}
	\begin{split}
		f\big(\bm{g},\ \bm{\epsilon}\big) &= f\big(\bm{g}|\ \bm{\epsilon}\big) \times f\big(\bm{\epsilon}\big) \\
	&= f\big(\bm{g}\big) \times f\big(\bm{\epsilon}\big) \\
	&= \mathcal{N}\big(\bm{g};\ \bm{\mu}_{OU}, \mathbf{\Sigma}_{OU}\big) \times \mathcal{N}\big(\bm{\epsilon};\ \bm{0}, \mathbf{\Sigma}_\mathcal{E}\big) \\
	&= \mathcal{N}\big(\bm{g};\ \bm{\mu}_{OU}, \mathbf{\Sigma}_{OU}\big) \times \mathcal{N}\big(\bm{z} - \bm{g};\ \bm{0}, \mathbf{\Sigma}_\mathcal{E}\big) \\
	&= \mathcal{N}\big(\bm{g};\ \bm{\mu}_{OU}, \mathbf{\Sigma}_{OU}\big) \times \mathcal{N}\big(\bm{g};\ \bm{z}, \mathbf{\Sigma}_\mathcal{E}\big)
	\end{split}
\end{align}
	
	Equations \ref{eq:MuG} and \ref{eq:SigmaG} follow from eq. \ref{eq:pdfG} and eq. 371, p. 42, section 8.1.8 ``Product of Gaussian densities'' in \citet{Petersen2012}.
\end{proof}

Importantly, equation \ref{eq:MuG} is the maximum likelihood estimate for $\bm{g}_v$, the pathogen effect on the trait, taking into account all available information - measured trait values, the pathogen phylogeny, and inferred POUMM parameters. This estimator is an inverse-variance weighted average of measured trait ($\bm{z}$) and information from the POUMM evolutionary model ($\bm{\mu}_{OU}$). In other words, $g$ will be closer to the measured trait value if the trait is not very heritable. If the trait is highly heritable, $g$ will be closer to the expected value under the POUMM, i.e. take more information from the phylogenetic relationships between infecting strains. 

Given the estimator we just derived for $\bm{g}$, we can now estimate $\bm{\epsilon}$, the trait value $without$ pathogen effects: 

\begin{equation}
	\hat{\bm{\epsilon}} = \bm{z} - Exp(\bm{g})
	\label{eq:EHat}
\end{equation}

We will use this value to try to improve upon standard GWAS methods in infectious disease.

\subsection*{A POUMM-based GWAS framework for infectious disease}

We propose to improve standard GWAS for infectious diseases by estimating and removing trait variability due to pathogen effects. Our new framework is as follows:

\begin{enumerate}
	\item Sample paired host genotypes, pathogen genome sequences, and trait values from a cohort.
	\item Construct a pathogen phylogeny using the pathogen genome sequences.
	\item Estimate the parameters of the POUMM based on the trait values and the pathogen phylogeny. This can be done with e.g. the R package POUMM \citep{Mitov2017a-POUMM}.
	\item Generate maximum-likelihood estimates for the pathogen and corresponding non-pathogen effects on the trait using equations \ref{eq:MuG} and \ref{eq:EHat}.
	\item Perform GWAS with only the non-pathogen effects on the trait as the response variable.
\end{enumerate}

\section*{Results}

\subsection*{Simulation study}

To test the theoretical best-case performance of our method, we simulated data under the POUMM and applied our framework to the simulated data. We parameterized our simulation scheme with the time-scale and other parameters of an HIV-1 outbreak in mind, with spVL as the trait of interest. 

We first simulated a phylogeny of 500 tips with exponentially distributed branch lengths and mean root-to-tip time of 0.14 substitutions per site per year as in \cite{Hodcroft2014}. Then, we simulated pathogen trait values $g$ along this phylogeny using the POUMM package in R \citep{Mitov2017a-POUMM}. For the simulation, we considered a range of pathogen heritability parameter values $H^2$, from 15 to 75\%, and a range of selection strength parameters values $\alpha$, from 0.1 to 60 time$^{-1}$. The intensity of stochastic fluctuations parameter $\sigma$ was determined based $H^2$ and $\alpha$ (a re-arrangement of equation \ref{eq:OUVar}, equation given in Table \ref{tab:sim-params}). As shown in Figure \ref{fig:sigma-vs-alpha}, higher $\alpha$ correspond to higher $\sigma$ to maintain constant $H^2$ under this parameterization. For each $H^2$ and $\alpha$ considered, we simulated recorded the simulated pathogen part of the trait value for each tip in the phylogeny.

We paired each tip's simulated pathogen trait value with a simulated host trait value. Simulated hosts had 100 genome positions. We sampled alleles (0, 1, or 2) for each position from a binomial distribution with probability 0.13. 10 random positions had an effect size of 0.2 on the trait and 10 had an effect size of -0.2. The other 80 positions had no effect on the trait. This parameterization produced roughly normally distributed host trait values centered at 0 with variance equal to 25\% of the total trait variance, which we constrained to 0.73 based on the variance in log spVL values measured by \cite{Mitov2018}. We used 25\% host heritability for spVL based on \cite{McLaren2015}.

Finally, we sampled an additional random environmental effect for each tip from a normal distribution centered at 0. The variance of this distribution was scaled based on the pathogen heritability of the trait, from 0 (no affect) in the scenario with 75\% pathogen heritability and 25\% host heritability to 0.44 in the scenario with 15\% pathogen heritability and 25\% host heritability. Figure \ref{fig:sim-design} shows a high-level schematic of this simulation framework and Table \ref{tab:sim-params} gives the value or expression for each parameter.

\subsubsection*{Estimator accuracy}

First, we evaluated how well our method estimated the additive host genetic effects from the simulated data. Additive host genetic effects represent an ideal (albeit unattainable) baseline for infectious disease GWAS. Figure \ref{fig:simulationResults}A shows that our method incorporating phylogenetic information can more accurately estimate these value compared to the trait value. To ensure a fair comparison, we scaled trait values to have the same mean, zero, as host genetic effects so as not to bias the root mean squared error (RMSE) by a constant factor. As shown in the supplemental material, we can calculate the expected RMSE using the scaled trait value across scenarios in our simulation scheme because the variance in the trait due to viral genetic effects and environmental effects is fixed. Thus, we expect the RMSE using the scaled trait value to be 0.74 across all simulation scenarios. By incorporating phylogenetic information, we can improve upon this error in scenarios where the trait is highly heritable, under low selection pressure, and with relatively moderate stochastic fluctuations compared to outbreak duration. Figure \ref{fig:simulationTraitExample} gives some intuition for how this correction works by contrasting simulated scenarios with high and low heritability and low selection strength/ low stochastic fluctuations. These parameter combinations mean trait values are phylogentically correlated at clustered tips in the phylogeny. This information is the basis from which our method estimates the pathogen and corresponding non-pathogen effects on the trait. Figure \ref{fig:tip-trait-correlation} shows in more detail how trait values are correlated at closely clustered tips when an infectious disease trait is highly heritable and selection strength is low/ stochastic fluctuations are low.

\subsubsection*{Theoretical GWAS improvement}

Next, we characterized the evolutionary scenarios under which our framework can actually improve GWAS power. We used the true positive rate (TPR) to evaluate the fraction of simulated causal host genetic variants we could recover as being significantly associated with the trait. We performed three different GWAS for each simulated dataset: the first represents an ideal in which we can exactly know and remove pathogen effects from trait values, the second is using our method to estimate this value and remove it, and the third represents a standard GWAS using the scaled trait value. Figure \ref{fig:simulationResults}B shows that our framework can improve the TPR in simulated scenarios where selection strength $<$ 10 time$^{-1}$ and heritability $>$ 45\%. If we were able to perfectly estimate and remove pathogen effects from a trait, the TPR would increase across all values of selection strength so long as the trait is more than marginally heritable. We estimate approximately 25\% to be the heritability threshold above which GWAS power is negatively impacted by pathogen effects. In summary, we show it is theoretically possible to improve GWAS power for heritable infectious disease traits by estimating and removing pathogen effects using information from the pathogen phylogeny.

% More accurate estimation of the non-viral part of spVL does not guarantee improved GWAS power. For example, if the viral part of spVL were constant across all viruses, association results (significance and effect size) would not be effected by a viral contribution to the trait. 
% Under our simulation set-up a standard GWAS would have $\approx$ 30\% power to detect a host variant explaining 1.25\% of the variability in spVL. This matches simulation results (figure \ref{fig:simulationResults}B, scaled spVL). 

\begin{figure}[H]
	\begin{center}
		\includegraphics[width=\linewidth]{figures/simulation_results}
		\caption{Results from the simulation study. We simulated host, pathogen, and environmental effects on a trait under the POUMM with different heritability (y-axis) and selection strength (x-axis) parameters. For each simulated dataset, we applied our method to estimate the non-pathogen effects and performed GWAS with these values. (A) shows the root mean squared error (RMSE) of our estimator (left) compared to un-corrected trait values, scaled by their mean (right) under each simulated evolutionary scenario. The RMSE is with reference to the true (simulated) host part of the trait values. Thus, more accurate estimates (lower RMSE) mean the trait value used for GWAS will be closer to the true host part of the trait value. (B) shows how GWAS power can improve given the true, simulated non-pathogen effect on spVL (left) and using our estimate for this value (middle) compared to using the scaled trait value (right). Each tile's color corresponds to the average value across 20 simulated datasets of 500 samples. The points highlight specific heritability and selection strength values from the \emph{A. thaliana}-\emph{X. arboricola} QDR analysis, HIV-1 spVL analysis, and four simulated scenarios that are presented in more detail in Figure \ref{fig:tip-trait-correlation}. RMSE = Root mean square error, TPR = True positive rate, QDR = quantitative disease resistance, spVL = set-point viral load.}
		\label{fig:simulationResults}
	\end{center}
\end{figure}

\begin{figure}[H]
	\begin{center}
		\includegraphics[width=\linewidth]{figures/example_simulation_trait_values}
		\caption{Simulated data from two evolutionary scenarios where a phylogenetic correction to trait values improves GWAS power (right side) and where it does not (left side). These examples correspond to two of the empty points in Figure \ref{fig:simulationResults}. (A) and (B) show total trait values for 12 randomly selected tips from the simulated phylogeny with pathogen heritability 15 and 75\%, respectively. Depending on the pathogen heritability, trait values are more or less correlated at clustered tips. (C) compares our method's estimate for the non-pathogen part of trait values (y-axis) with true simulated host trait values (x-axis) with pathogen heritability 15 and 75\%. The solid line is the y=x line. Selection strength was fixed to 0.1 time $^{-1}$ for both scenarios and all other parameters were fixed as in the full simulation study.}
		\label{fig:simulationTraitExample}
	\end{center}
\end{figure}

\begin{figure}[H]
\begin{center}
\includegraphics[width = \linewidth]{figures/tip_trait_correlation.png}
	\caption{Correlations between trait values in pairs of tips in four simulated scenarios. These examples correspond to the four empty points in Figure \ref{fig:simulationResults}. Correlations are calculated for pairs of tips binned by phylogenetic distance (into deciles) across the 20 replicate simulations for each of the four evolutionary scenarios. Trait values are only noticeably correlated for closely clustered tips under the scenario with high pathogen heritability and low selection strength/ low stochastic fluctuations (upper left facet).}
	\label{fig:tip-trait-correlation}
	\end{center}
\end{figure}

\subsection*{Application to HIV-1 set-point viral load}

We applied our framework to empirical data from two different host-pathogen systems. First, we used data collected by the Swiss HIV Cohort Study (SHCS) from 1,392 individuals in Switzerland infected with HIV-1 subtype B between 1994 and 2018. The SHCS provided viral load measurements, $pol$ gene sequences, and human genotype data for these individuals. We followed the method outlined above to estimate the pathogen and non-pathogen effects on spVL for the cohort from these data. Figure \ref{fig:spvl-calc-comparison} shows the calculated (total) spVL values, which vary between approximately 1 and 6 log copies/mL in the cohort.  We estimated spVL heritability in this cohort to be 45\% (95\% highest posterior density, HPD, 24 - 67\%) and selection strength to be 58 time$^{-1}$ (95\% HPD 19 - 95) (Figure \ref{fig:poumm-parameters}, Table \ref{tab:POUMM-parameters-our-estimates}). To put these values into the context of our simulation study, they are shown as points on Figure \ref{fig:simulationResults}. The highest expected correlation in trait values between any two tips in the HIV-1 phylogeny under the POUMM was 0.45. However, Figure \ref{fig:spvl-on-tree} shows that this trait is not obviously phylogenetically structured in the cohort in general, despite high heritability. Finally, figure \ref{fig:trait-comparison} shows that the estimated non-pathogen effects on spVL correlate quite strongly with total spVL. 

We compared our proposed GWAS framework with a more standard approach by performing two different GWAS on the same SHCS human genotypes. In the (i) ``GWAS with standard trait value'' we used the total trait value, our calculated spVL values, as the GWAS response variable. In the (ii) ``GWAS with estimated non-pathogen part of trait'' we used our estimates for the non-pathogen effects on spVL. Figure \ref{fig:gwas-results}A shows that results are qualitatively similar between the two GWAS. Q-Q plots show the distribution of p-values are very similar as well (Figure \ref{fig:qq-plots}). Figure \ref{fig:gwas-results}B shows how the strength of association changed for some variants in the MHC and \emph{CCR5} regions. Taking into account phylogenetic information slightly decreased association strength for most variants in the \emph{CCR5} region. Association strength increased for some variants in the MHC, for example, SNP rs9265880 had the greatest increase in significance in the MHC region, from a p-value of $3.5 \times 10^{-07}$ to $7.7 \times 10^{-09}$. However, the top-associated variants in the MHC and \emph{CCR5} regions were consistent regardless of the GWAS response variable used (Table \ref{tab:gwas-top-associations}). Finally Table \ref{tab:comp-gwas-mclaren-snps} shows how our GWAS results compare for the two top-associated SNPs identified by \citet{McLaren2015}, who performed the largest standard GWAS for HIV spVL to date. In summary, there are no clear patterns that point to new regions of association in the human genome with spVL when we take into account the pathogen phylogeny.

\begin{figure}[H]
	\begin{center}
		\includegraphics[width=\linewidth]{figures/gwas_results}
		\caption{Results from comparative GWAS. (A) shows association p-values for the same host variants from the SHCS cohort in GWAS with two different response variables. On the left, we used unmodified (total) spVL values. On the right, we used our estimates for the non-pathogen effects on spVL. The alternating shades correspond to different chromosomes. (B) compares the strength of association for variants in the \emph{CCR5} and MHC regions between the two GWAS (positions 45.4 - 47Mb on chromosome 3 and 29.5 - 33.5Mb on chromosome 6 for the \emph{CCR5} and MHC, respectively). Base positions are with reference to genome build GRCh37. The color of each point represents the difference in -log$_{10}$ p-value between the two GWAS. Red means taking into account phylogenetic information decreased the strength of association and blue means it increased it. The dashed lines show genome-wide significance at p = $5 \times 10^{-8}$.}
		\label{fig:gwas-results}
	\end{center}
\end{figure}

\begin{table}[H]
\centering
\caption{Top association results from \citet{McLaren2015} compared to results from this study. Results from this study are for host variants from the SHCS in GWAS with two different response variables. ``Standard trait value'' means we used the unmodified (total) spVL value and ``Estimated non-pathogen part of trait'' means we used our estimates for the non-pathogen effects on spVL.}
\begin{tabularx}{\linewidth}{p{1cm}p{1.8cm}p{2.4cm}p{1.8cm}p{2cm}p{1.8cm}p{2cm}} % {lrrrrrr} {llclcccc}
\hline
& & \makecell{McLaren \\ et al.}  & \multicolumn{2}{l}{\makecell{Standard \\ trait value}} & \multicolumn{2}{l}{\makecell{Estimated non-pathogen \\ part of trait}} \\ 
  \hline
 Region & Variant & p-value & Effect size & p-value & Effect size & p-value \\
  \hline
    MHC & rs59440261 & $2.0 \times 10^{-83}$ & -0.4 & $3.3 \times 10^{-11}$ & -0.22 & $2.6 \times 10^{-10}$ \\ 
    \emph{CCR5} & rs1015164 & $1.5 \times 10^{-19}$ & 0.15 & $7.5 \times 10^{-7}$ & 0.078 & $8.5 \times 10^{-6}$ \\ 
   \hline
\end{tabularx}
\label{tab:comp-gwas-mclaren-snps}
\end{table}

\subsection*{Application to the \emph{A. thaliana}-\emph{X. arboricola} pathosystem}

Next, we applied our method to data collected from the \emph{A. thaliana}-\emph{X. arboricola} pathosystem by \cite{Wang2018Two-wayGenomes}. \cite{Wang2018Two-wayGenomes} performed a fully-crossed experiment in which they infected genetically diverse \emph{A. thaliana} accessions with genetically diverse strains of the phytopathogenic bacteria \emph{X. arboricola}. They scored quantitative disease resistance (QDR) on a scale of 0 (resistant) to 4 (susceptible) for up to four infected leaves for three replicates of each \emph{A. thaliana}-\emph{X. arboricola} pairing. Figure \ref{fig:qdr-on-tree} shows the inferred \emph{X. arboricola} pathogen phylogeny. Mean QDR calculated across all \emph{A. thaliana} pairings and all replicates for each \emph{X. arboricola} strain was generally low, varying between 0.11 for strain NL\_P126 and 0.78 for strain FOR\_F21. Fitting the POUMM yielded very low selection strength $\alpha$ and intensity of stochastic fluctuations  $\sigma$ parameter estimates (posterior mean 0.03 with 95\% HPD 0.0 - 0.05 and 0.03 with 95\% HPD 0.0 - 0.06, respectively; Table \ref{tab:POUMM-parameters-our-estimates-qdr}). These values deviated significantly from the respective priors (Figure \ref{fig:poumm-parameters-qdr}). Heritability, on the other hand, was quite uncertain (posterior mean 0.33 with 95\% HPD 0.0 - 0.77; Table \ref{tab:POUMM-parameters-our-estimates-qdr}). The posterior mean selection strength and heritability values are also shown in the context of the simulation study as points on Figure \ref{fig:simulationResults}.

Given the posterior mean estimates for the POUMM parameters, expected correlation in trait values between tips were very low (maximum value $3.2 \times 10^{-12}$ compared to maximum value of 0.45 in the HIV-1 spVL application). Thus, the phylogeny is not very informative for a trait value correction. Indeed, mean QDR values for each \emph{X. arboricola} strain calculated across all \emph{A. thaliana} pairings and all replicates is simply scaled by our method (Figure \ref{fig:trait-comparison-qdr}).

\section*{Discussion}
% State the importance of your findings
In this paper, we presented a new phylogeny-aware GWAS framework to correct for heritable pathogen effects on infectious disease traits. By using information from the pathogen phylogeny, we show it is possible to improve GWAS power to detect host genetic variants associated with a disease trait. This should help us better understand which host factors are broadly protective against a disease versus which increase susceptibility or disease severity.

% Contextualize the study & address limitations
Our method relies on the POUMM, a model of continuous trait evolution that accounts for heritable and non-heritable effects on a trait, as well as selection. We applied this model to two different host-pathogen systems where paired host and pathogen genetic data was generated alongside a measure of pathogen virulence. First, we fit the POUMM to set-point viral load data from individuals living with HIV in Switzerland. We estimated HIV-1 spVL heritability to be 45\% (95\% HPD 24 - 67\%) in this cohort. Compared to previous studies, this estimate is at the higher end (see \citet{Mitov2018} and references therein). Also using the POUMM, \citet{Bertels2018} estimated a spVL heritability of 29\% (N = 2014, CI 12 - 46\%) from the same cohort and \citet{Blanquart2017} estimated 31\% (N = 2028, CI 15 - 43\%) from a pan-European cohort. We note that our sample size (N = 1493 individuals) is smaller than in these other studies. This might be because we restricted samples based on having $pol$ gene sequences with at least 750 non-ambiguous bases. Our aim was to reconstruct a high-quality phylogeny, since the POUMM does not account for phylogenetic uncertainty and the POUMM parameter estimates are key to our downstream trait-correction method. Although our heritability estimate is rather high, the confidence interval largely overlaps that of other studies and we note that estimating heritability per se was not our primary focus.

For comparison, we also fit the POUMM to quantitative disease resistance measurements from \emph{A. thaliana} infected with the phytopathogenic bacteria \emph{X. arboricola}. We estimated \emph{X. arboricola} virulence heritability to be 33\% (95\% HPD 0 - 77\%). \citep{Wang2018Two-wayGenomes} originally estimated a QDR heritability of 44\% in this dataset, falling within the wide range of our estimate. We note that \cite{Wang2018Two-wayGenomes} used a linear mixed model in which the experimental unit is QDR scored on individual leaves, whereas our estimate is based on much coarser binning of QDR scores into a mean score across all leaves on all host accessions and all replicates (N = 22). Furthermore, the QDR score trait values were not truly continuous (scores were measured on an integer scale from 0 to 4). Thus, these data partially violate the assumptions of the POUMM. Compared to HIV-1, we estimate very low selection strength for virulence in \emph{X. arboricola}. As \cite{Wang2018Two-wayGenomes} explain, \emph{X. arboricola} strains with differing virulence can co-inhabit populations of \emph{A. thaliana}. This might also point to low selection on \emph{X. arboricola} virulence. Furthermore, expected correlation in virulence between related strains of \emph{X. arboricola} was smaller than for HIV-1.

The main novelty of our approach is to use these POUMM parameter estimates to correct infectious disease trait values prior to performing GWAS, thereby estimating and removing pathogen effects. In simulations, we show that when trait heritability amongst infection partners is greater than approximately 25\%, GWAS power to detect host genetic variants associated with the same trait is reduced. Our method can correct for this effect in certain evolutionary scenarios by using information from the full pathogen phylogeny. Based on our simulation results, our method is anticipated to be very useful for disease traits that are highly heritable from donor to recipient and  maintain a high correlation between sampled individuals. In simulations, we showed this is the case when pathogen heritability is high, selection strength is low, and trait values are not subject to strong stochastic fluctuations. It is possible that traits under strong selection pressure would also be highly heritable and phylogenetically correlated, for instance antimicrobial resistance. However, such traits might violate the POUMM assumption that trait values vary as a random walk if they are caused by few mutations of strong affect. In summary, cohort-level, phylogenetically structured differences in the measured trait value are necessary for our approach to outperform state of the art methods. 

Given our estimates for trait heritability and selection strength on HIV-1 spVL and \emph{A. thaliana} quantitative disease resistance to \emph{X. arboricola}, our simulation results reveal that we cannot expect a significant improvement in GWAS power (Figure \ref{fig:simulationResults}). Certain pairs of samples in the HIV-1 cohort were indeed expected to have phylogenetically correlated spvL values (maximum expected correlation between any two samples was 0.45). However, the overall effect on GWAS is small. For HIV-1 spVL, our phylogenetic correction slightly decreases p-values for variants in $CCR5$ and slightly decreases some and increases other p-values for variants in the MHC (Figure \ref{fig:gwas-results}B). Simulations show we shouldn't expect a net p-value decrease, but our simulations represent an ideal scenario since we simulate under the POUMM. In real life, un-modeled evolutionary pressures like drug treatment and host-specific HLA alleles might cause the reduced p-values. However, the overall picture is consistent between the two GWAS (Figure \ref{fig:gwas-results}A). Therefore, we conclude that GWAS for host determinants of HIV-1 subtype B spVL is robust to our correction for pathogen effects. For \emph{A. thaliana} QDR, inferred phylogenetic correlations are too weak to justify our phylogenetic correction (maximum expected correlation between strains was $3.2 \times 10^{-12}$). \cite{Wang2018Two-wayGenomes} found no significant \emph{A. thaliana} variants associated with QDR using a linear mixed model jointly accounting for host genetic effects, pathogen genetic effects, and interaction effects. As with HIV-1 spVL, our results do not challenge this previous finding.

Our approach is envisioned to be complementary to previously developed methods for infectious disease GWAS. First, it provides additional information on the evolutionary dynamics of the trait in the pathogen population. Then, it is a convenient pre-processing step for GWAS because it simply produces an alternate response variable for GWAS association tests. As in the \emph{A. thaliana}-\emph{X. arboricola} application, fitting the POUMM may reveal that expected phylogenetic correlations between samples are not strong enough to justify using our method to correct trait values in a GWAS. In this case, one may wish to use a linear mixed model as in \citep{Wang2018Two-wayGenomes}, where the pathogen effect is co-estimated as a random effect. In cases where a correction can be estimated and applied, it is still possible to use any of the number of previously developed GWAS models for the actual association testing (we used a linear model approach implemented in PLINK \citep{Chang2015}, though a more advanced method would be to use a linear mixed model with host ancestry as a random effect). Finally, additional model complexity can be added to the GWAS association tests. For instance, our method does not account for co-infection, which might add additional variance to trait values and decrease GWAS power. In this case, one could add co-infection status as a covariate in the GWAS association test to account for this variable. 

Our method relies on the freely available R package POUMM \citep{Mitov2017a-POUMM}, which scales to trees of up to 10,000 tips \citep{Mitov2019}. All code for the simulations and HIV spVL analysis presented in this study is available on the project GitHub at https://github.com/cevo-public/POUMM-GWAS. Future applications of our method might investigate other clinically significant disease traits and outcomes that are affected by both host and pathogen genetic factors, for instance Hepatitis B Virus-related hepatocellular carcinoma \citep{An2018HostCarcinoma}, Hepatitis C treatment success \citep{Ansari2017Genome-to-genomeVirus}, and susceptibility to or severity of certain bacterial infections, e.g. \cite{Messina2016ImpactInfections, Donnenberg2015BacterialStudies}. Transcriptomic data has also previously been modeled as an evolving phenotype using an Ornstein-Uhlenbeck model \citep{Rohlfs2014ModelingVariation}. Thus, one could also estimate pathogen effects on host gene expression. In particular, it will be of interest to understand the evolutionary dynamics of these traits in the respective pathogen populations.

% Summarize the paper
In summary, we argue that infectious disease GWAS should take the pathogen phylogeny into account when searching for host determinants of a disease trait. We give a practical threshold for identifying when GWAS suffer from pathogen effects (heritability of the trait amongst infection partners $>$ 25\%) and provide a method that can help in scenarios where trait values are highly heritable and phylogenetically-structured amongst members of a cohort.

\section*{Materials and Methods}

\subsection*{Simulation model}

Whenever possible, we tried to parameterize our simulation model for spVL using empirical data. We set the total variance in spVL to 0.73 log copies$^2$ mL$^{-2}$ based on UK cohort data \citep{Mitov2018}. Other studies have estimated slightly lower values though (Table \ref{tab:spVLmeasurements}). After allotting 25\% of this variance to the host part of spVL $h$ based on results by \citet{McLaren2015}, we partitioned the remaining variance between the viral part $g$ and the environmental part $\epsilon$ in different ratios to assess estimator performance across a range of spVL heritabilities. $h$ was simulated as the sum of contributions from 20 causal host genetic variants, 10 of which had an effect size of 0.2 log copies mL$^{-1}$ and 10 of which had an effect size of -0.2 log copies mL$^{-1}$. Host genetic variants were generated from a binomial distribution with probability $p$ calculated such that $h$ had the appropriate variance (see Table \ref{tab:sim-params}). We generated a random viral phylogeny with branch lengths on the same time scale as a previously inferred UK cohort HIV tree \citep{Hodcroft2014} using the R package ape \citep{Paradis2018}. $g$ was simulated by running an OU process along the phylogeny using the R package POUMM \citep{Mitov2017a-POUMM} and sampling values at the tips. For the OU parameters $\theta$ and $g_{0}$ we used 4.5 log copies mL$^{-1}$ based on fitting the same model to SCHS data (Table \ref{tab:POUMM-parameters-our-estimates}). This is similar to values previously inferred for HIV (Table \ref{tab:POUMMparams-prev-estimated}). To assess our estimator's performance under a range of evolutionary scenarios, we co-varied the heritability $H^2$ and selection strength $\alpha$ parameters. The intensity of random fluctuations $\sigma$ was determined based on these parameters (Table \ref{tab:sim-params}, Figure \ref{fig:sigma-vs-alpha}). Finally, the environmental part of spVL $\epsilon$ was generated from a normal distribution with mean 0.  For a full graphical model representation of the simulation scheme, see Figure \ref{fig:sim-design}. 

We performed GWAS on the simulated data using a linear association model as implemented in the ``lm'' function in R. For each simulated dataset, we performed three association tests: (i) using the true (simulated) non-pathogen part of the trait (host + environmental parts), (ii) using the estimated non-pathogen part of the trait according to the method presented in this paper, and (iii) using the total trait value, scaled by its mean. We assessed the significance of each associations at a significance level of 0.05 with a Bonferroni correction for multiple testing. For our main results (Figure \ref{fig:simulationResults}) we simulated 20 truly associated variants, as described above. To also check the false positive rate (FPR), we re-ran the simulations with an additional 80 non-associated variants. Across all the association tests in this second simulation setup (7 $H^2$ levels $\times$ 10 $\alpha$ levels $\times$ 100 variants $\times$ 20 replicates per scenario = 140,000 association tests), FPR was 0.0005 using the true (simulated) non-pathogen part of the trait, 0.0005  using the estimated non-pathogen part of the trait, and 0.0006 using the scaled total trait value. These rates are comparable to the expected FPR of 0.0005 at significance level 0.05 corrected for 100 tests. Given the stricter correction for multiple testing in this second simulation setup, the TPR decreased significantly across all three GWAS response variables used.

\begin{table}[H]
	\caption{Simulation model parameters. For a full graphical model representation of the simulation scheme, including how these parameters are related, see Figure \ref{fig:sim-design}.}
	\begin{tabularx}{\linewidth}{p{1.5cm}ll} \toprule 
		Variable & Expression & Definition \\ \midrule 
		$\sigma^2_z$ &  0.73 log copies$^{2}$/mL$^{2}$ & Total spVL variance \\ 
		$H^2_h$ & 0.25 & Host heritability of spVL \\
		$H^2_{\bar{t}}$ & varied & Viral heritability of spVL at $\bar{t}$ \\
		$\sigma^2_{h}$ & $\sigma_{h}^2 = 0.25*\sigma_z^2$ & Variance in host part of spVL \\
		$\sigma^2_{p}(\bar{t})$ & $\sigma^2_{g_{v}}(\bar{t}) = H^2_{\bar{t}}*\sigma_z^2$ & Variance in viral part of spVL at $\bar{t}$ \\ 
		$\sigma^2_{\epsilon}$ & $\sigma_{\epsilon}^2 = \sigma_z^2 - \sigma^2_{p} - \sigma^2_{h}$ & Variance in environmental part of spVL \\ 
		$\bar{t}$ & 0.14 substitutions site$^{-1}$ yr$^{-1}$ & Mean root-tip time in viral phylogeny \\ 
		$\bm{g}_v$ & $\bm{g}_v \sim Norm(\bm{\mu}_{OU}, \boldsymbol{\Sigma}_{OU})$ & Viral part of spVL for all individuals \\ 
		$\theta$ & 4.5 log copies/mL & Optimal spVL value \\
		$g_{0}$ & 4.5 log copies/mL & $g$ at the root of the phylogeny  \\ 
		$\alpha$ & varied & Selection strength of OU process \\ 
		$\sigma$ & $\sigma = \sqrt{\frac{2\alpha\sigma^2_{g_{v}}(\bar{t})}{1 - exp(-2\alpha\bar{t})}}$ & Time-unit standard deviation of OU process \\ 
		$\Psi$ & branch lengths $\sim Exp(\bar{t})$ & Viral phylogeny \\ 
		$g_{h_i}$ & \makecell{$g_{h_i} = \delta \sum_{j = 1}^{j = M/2}{G_{ij}} - \\ \delta \sum_{j = M/2}^{j = M}{G_{ij}}$} & Host part of spVL for individual $i$ \\ 
		$G_{N \times M}$ & \makecell{$G_{ij} \sim Binom(2, p)\ \\ \forall i \in {1...N}, \forall j \in {1...M}$} & Host genotype matrix \\
		$p$ & $p = \frac{1}{2} - \sqrt{\frac{1}{4} - \frac{H_{h}^2\sigma^2_z}{2\delta^2M}}$ &  Host variant allele frequency \\ 
		$\delta$ & 0.2 & Host variant effect size \\ 
		$M$ & 20 & Number of causal host variants \\ 
		$\epsilon_i$ & $ \epsilon_i \sim Norm(0, \sigma^2_{\epsilon})$ & Environmental part of spVL for individual $i$ \\ 
		$N$ & 500 & Number of simulated samples \\ \bottomrule 
	\end{tabularx}
	\label{tab:sim-params}
\end{table}

\subsection*{Swiss HIV-1 data}

Human genotypes, viral load measurements, and HIV-1 $pol$ gene sequences from HIV-1 positive individuals were all collected in the context of other studies by the Swiss HIV Cohort Study (SHCS) (www.shcs.ch, \citet{Schoeni-Affolter2010, Scherrer2021CohortSHCS}). All participants were HIV-1–infected individuals 16 years or older and written informed consent was obtained from all cohort participants. The anonymized data were made available for this study after the study proposal was approved by the SHCS. 

For phylogenetic inference, we retained sequences from 1,493 individuals with non-recombinant subtype B $pol$ gene sequences of at least 750 characters and paired RNA measurements allowing for calculation of spVL, as well as 5 randomly chosen subtype A sequences as an outgroup. We used MUSCLE version 3.8.31 \citep{edgar_muscle:_2004} to align the $pol$ sequences with --maxiters 3 and otherwise default settings. We trimmed the alignment to 1505 characters to standardize sequence lengths. We used IQ-TREE version 1.6.9 \citep{Nguyen2014} to construct an approximate maximum likelihood tree with -m GTR+F+R4 for a general time reversible substitution model with empirical base frequencies and four free substitution rate categories. Otherwise, we used the default IQ-TREE settings. After rooting the tree based on the subtype A samples, we removed the outgroup. Viral subtype was determined by the SHCS using the REGA HIV subtyping tool version 2.0 \citep{DeOliveira2005}. We calculated spVL as the arithmetic mean of viral RNA measurements made prior to the start of antiretroviral treatment. For a comparison of several different filtering methods, see Figure \ref{fig:spvl-calc-comparison}. 

For GWAS, we retained data from 1,392 of the 1,493 SHCS individuals with European ancestry who were not closely related to other individuals in the cohort (Table \ref{tab:sample-filtering}). These were 227 females and 1165 males. Ancestry was determined by plotting individuals along the three primary axes of genotypic variation from a combined dataset of SHCS samples and HapMap populations (Figure \ref{fig:PCA}). Kinship was evaluated using PLINK version 2.3 \citep{Chang2015}; we used the --king-cutoff option to exclude one from each pair of individuals with a kinship coefficient $>$ 0.09375. Initial host genotyping quality control and imputation were done as in \cite{Thorball2021GeneticLymphoma}. Subsequent genotyping quality control was performed using PLINK version 2.3 \citep{Chang2015}. We used the options --maf 0.01, --geno 0.01, and --hwe 0.00005 to remove variants with minor allele frequency less than 0.01, missing call rate greater than 0.05, or Hardy-Weinberg equilibrium exact test p-value less than 5x10$^-5$. After quality filtering, approximately 6.2 million genetic variants from the 1,392 individuals were retained for GWAS (Table \ref{tab:variant-filtering}). 

\subsection*{\emph{A. thaliana}-\emph{X. arboricola} data}
\emph{A. thaliana} and \emph{X. arboricola} genotyping and quantitative disease resistance (QDR) measurements are described in detail in \cite{Wang2018Two-wayGenomes}. Briefly, \cite{Wang2018Two-wayGenomes} infected different \emph{A. thaliana} host accessions with different \emph{X. arboricola} pathogen strains in a fully-crossed experimental design. They infected up to 4 leaves on each of three biological replicates for each host-pathogen pairing. Then, they scored QDR for each leaf on a scale of 0 (resistant) to 4 (susceptible). We downloaded the genotype matrix with allele dosage of 33,610 SNPs for the 22 \emph{X. arboricola} pathogen strains generated by \cite{Wang2018Two-wayGenomes} from their supplemental material. We additionally downloaded a VCF file with allele dosage of 12,883,854 SNPs for the different \emph{A. thaliana} accessions from the 1001 Genomes project \citep{Alonso-Blanco20161135Thaliana}. QDR measurements were provided directly by the  \cite{Wang2018Two-wayGenomes} authors.

For phylogenetic inference, we used the ``dist.gene'' and ``nj'' functions from the ape package in R to construct a pairwise genetic distance matrix and then a neighbor-joining tree from the \emph{X. arboricola} pathogen genotype matrix. The inferred tree topology (Figure \ref{fig:qdr-on-tree}) closely matches the hierarchical clustering presented in \citep{Wang2018Two-wayGenomes} generated using the unweighted pair group method with arithmetic mean (UPGMA). Compared to UPGMA, the neighbor-joining method we used relaxes the assumptions of a strict molecular clock and sampling all at the same time-point. For the trait value to fit the POUMM, we calculated mean QDR across all leaves infected on all hosts for each \emph{X. arboricola} strain.

\subsection*{POUMM parameter inference}

We used the R package POUMM version 2.1.6 \citep{Mitov2017a-POUMM} to infer the POUMM parameters $g_0, \alpha, \theta, \sigma, \text{ and }\sigma_e$ from the HIV-1 and \emph{X. arboricola} phylogenies and associated spVL and QDR trait values. The Bayesian inference method implemented in this package requires specification of a prior distribution for each parameter. For HIV-1 spVL, we used the same, broad prior distributions as in \citet{Mitov2018}, namely: $g_{0} \sim \mathcal{N}(4.5,\ 3)$, $\alpha \sim Exp(0.02)$, $\theta \sim \mathcal{N}(4.5,\ 3)$, $H^2_{\bar{t}} \sim \mathcal{U}(0,\ 1)$, and $\sigma^2_e \sim Exp(0.02)$. For \emph{X. arboricola} QDR, we modified the $g_0$ and $\theta$ priors to match the empirical mean and standard deviation of QDR trait values in the dataset: $g_{0} \sim \mathcal{N}(0.4,\ 0.2)$ and $\theta \sim \mathcal{N}(0.4,\ 0.2)$. We ran two MCMC chains for 4x10$^6$ samples each with a target sample acceptance rate of 0.01 and a thinning interval of 1000 for both analyses. The first 2x10$^5$ samples of each chain were used for automatic adjustment of the MCMC proposal distribution. Figures \ref{fig:poumm-parameters} and \ref{fig:poumm-parameters-qdr} show the posterior distributions for inferred parameters for HIV-1 spVL and \emph{X. arboricola} QDR, respectively. Table \ref{tab:POUMM-parameters-our-estimates} gives the posterior mean values used for subsequent calculations.

\subsection*{Phylogenetic trait correction}
We corrected HIV-1 spVL and \emph{X. arboricola} mean QDR trait values using the method described in this paper. For each  individual, we estimated the pathogen part of the trait value using equation \ref{eq:MuG} and the corresponding non-pathogen part using equation \ref{eq:EHat}. This is implemented in the function ``POUMM:::gPOUMM'' in the R package POUMM. In the HIV-1 case, each sample corresponds to one HIV-1 strain with one spVL value. In the \emph{X. arboricola} case, each sample corresponds to one \emph{X. arboricola} strain and the mean QDR score for that strain. To calculate the expected correlation in trait values between tips in the pathogen phylogeny, we used the function ``covVTipsGivenTreePOUMM'' in the same package. For the POUMM parameters $\alpha$, $\sigma$, $\theta$, and $\sigma_e$, we used the posterior mean estimates generated as described above. All the code used to implement the method is available at https://github.com/cevo-public/POUMM-GWAS.

\subsection*{Association testing}
We performed two GWAS using the same human genotype data from the SHCS. For the first ``GWAS with standard trait value'' we used the total (uncorrected) trait values ($\bm{z}$) as the response variable for association testing, replicating a standard GWAS set-up. For the second ``GWAS with estimated non-pathogen part of trait'' we replaced total trait values with the estimated non-pathogen component of the trait ($\hat{\bm{e}}$) as the response variable. Association testing was performed using a linear association model in PLINK version 2.3 \citep{Chang2015} with sex and the top 5 principle components of host genetic variation included as covariates. The sex and principle components covariates were included to reduce residual variance in spVL and control for confounding from host population structure, respectively. 

\subsection*{Data availability}
The simulated data underlying this article can be re-generated using the code available on the project GitHub at https://github.com/cevo-public/POUMM-GWAS. The HIV pathogen genome sequences, clinical data, and human genotypes cannot be shared publicly due to the privacy of individuals who participated in the cohort study. The data may be shared on reasonable request to the Swiss HIV Cohort Study at http://www.shcs.ch. The \emph{X. arboricola} pathogen genotypes are available in the supplemental material of \citep{Wang2018Two-wayGenomes}, the \emph{A. thaliana} host genotypes are available at https://1001genomes.org/, and the \emph{A. thaliana}-\emph{X. arboricola} QDR measurements are available on request to the authors of \citep{Wang2018Two-wayGenomes}.

\section*{Acknowledgments}

This work was supported by ETH Zurich. We thank the patients who participate in the SHCS; the physicians and study nurses for excellent patient care; A. Scherrer,  E. Mauro, and K. Kusejko from the SHCS Data Centre for data management; and D. Perraudin and M. Amstad for administrative assistance. We thank Joy Bergelson for sharing the \emph{A. thaliana}-\emph{X. arboricola} QDR measurements. We also thank Michael Landis, who shared a LaTeX template for graphical model drawing.
 
The members of the SHCS are:
Abela I, Aebi-Popp K, Anagnostopoulos A, Battegay M, Bernasconi E, Braun DL, Bucher HC, Calmy A, Cavassini M, Ciuffi A, Dollenmaier G, Egger M, Elzi L, Fehr J, Fellay J, Furrer H, Fux CA, Günthard HF (President of the SHCS), Hachfeld A, Haerry D (deputy of "Positive Council"), Hasse B, Hirsch HH, Hoffmann M, Hösli I, Huber M, Kahlert CR (Chairman of the Mother & Child Substudy), Kaiser L, Keiser O, Klimkait T, Kouyos RD, Kovari H, Kusejko K (Head of Data Centre), Martinetti G, Martinez de Tejada B, Marzolini C, Metzner KJ, Müller N, Nemeth J, Nicca D, Paioni P, Pantaleo G, Perreau M, Rauch A (Chairman of the Scientific Board), Schmid P, Speck R, Stöckle M (Chairman of the Clinical and Laboratory Committee), Tarr P, Trkola A, Wandeler G, Yerly S.

The Swiss HIV Cohort Study is supported by the Swiss National Science Foundation (grant #201369), by SHCS project #858 and by the SHCS research foundation. Furthermore, the SHCS drug resistance database is supported by the Yvonne Jacob Foundation (to HFG). The data are gathered by the Five Swiss University Hospitals, two Cantonal Hospitals, 15 affiliated hospitals and 36 private physicians (listed in http://www.shcs.ch/180-health-care-providers).
% \end{doublespace}
\end{linenumbers}

\bibliographystyle{manuscript/my_natbib}
\bibliography{manuscript/references_synced}

%Additional ideas for supplemental materials:
%\begin{itemize}
%	\item Table of genotype samples by project (genotyping chip, number of samples, spVL distribution)
%	\item Link to github with all the code used to run simulations and generate figures
%	\item Example spVL plots
%	\item Proof of robustness of method to inferred tree -- NA because I just re-ran the _whole_ thing and it's not that different
%\end{itemize}

\newpage
\startsetups
    \setcounter{figure}{0}
    \renewcommand{\thefigure}{S\arabic{figure}}
    \setcounter{table}{0}
    \renewcommand{\thetable}{S\arabic{table}}
\stopsetups
\section*{Supplemental Material}\label{supp}
\beginsupplement
\setcounter{page}{1}

\begin{doublespace}

\subsection*{Expected results from simulations}

Here we show the root mean square error (RMSE) of $\bm{z} - \bm{\bar{z}}$ as an estimate for $\bm{h}$ should be $\approx$ 0.74 in our simulation scheme. First we write the expression for the RMSE:

\begin{equation}
    RMSE = \sqrt{\frac{\sum_i^N(z - \bar{z} - h)^2}{N}}
\end{equation}

Note that $z - \bar{z}$ differs from $h$ due to a viral effect and an environmental effect. So the term inside the square root equals the combined variance of these two effects:

\begin{equation}
    RMSE = \sqrt{\sigma^2_{p} + \sigma^2_\epsilon}
\end{equation}

We can calculate the variance due to these two effects because the total variance in spVL $\sigma^2_z$, and the fraction of the total variance due to host genetic effects, $\sigma^2_{h}$, are fixed parameters in our simulation scheme.

\begin{equation}
\begin{split}
    \sigma^2_{h} + \sigma^2_{p} + \sigma^2_\epsilon = \sigma^2_z \\
    0.25 * \sigma^2_{z} + \sigma^2_{p} + \sigma^2_\epsilon = \sigma^2_z \\
    \sigma^2_{p} + \sigma^2_\epsilon = 0.75 * \sigma^2_z \\
    \sigma^2_{p} + \sigma^2_\epsilon = 0.75 * 0.73 \\
    \sigma^2_{p} + \sigma^2_\epsilon = 0.55
\end{split}
\end{equation}

Therefore, we can expect the RMSE for $\bm{z} - \bm{\bar{z}}$ as an estimate for $\bm{h}$ to be around $\sqrt{0.55} \approx 0.74$. 

\newpage

\begin{figure}[H]
	\begin{tikzpicture}
	
	% Variance 
	\begin{scope}[xshift = 0cm, yshift = 0cm]
	\node[anchor = west] (alabel) at (-1, 4) {\textbf{A} Variance partitioning};
	\node[constnode] (H2h) at (0, 0) {0.25};
	\node[constnode] (varz) at ($(H2h) + (1.8, 0)$) {$\sigma^2_z$};
	\node[constnode] (H2) at ($(H2h) + (3.6, 0)$) {\textcolor{red}{$H^2_{\bar{t}}$}};
	\node[detnode] (varh) at ($(H2h) + (0.9, 1.5)$) {$\sigma^2_{h}$};
	\node[detnode] (varv) at ($(varz) + (0.9, 1.5)$) {\tiny $\sigma^2_{g_{v}}(\bar{t})$};
	\node[detnode] (vare) at ($(varh) + (0.9, 1.5)$) {$\sigma^2_{\epsilon}$};
	\draw[taro] (H2h) -- (varh);
	\draw[taro] (varz) -- (varh);
	\draw[taro] (varz) -- (varv);
	\draw[taro] (H2) -- (varv);
	\draw[taro] (varh) -- (vare);
	\draw[taro] (varv) -- (vare);
	\draw[taro] (varz) -- (vare);
	\end{scope}
	
	% Virus
	\begin{scope}[xshift = 6cm, yshift = -0.5cm]
	\node[anchor = west] (blabel) at (-1, 4.5) {\textbf{B} Generating viral effects};
	\node[constnode] (theta) at (0, 1.5) {$\theta$};
	\node[constnode] (g0) at (1.8, 1.5) {$g_{0}$};
	\node[constnode] (alpha) at (3.6, 1.5) {\textcolor{red}{$\alpha$}};
	\node[detnode] (sigma) at (5.4, 1.5) {$\sigma$};
	\node[constnode] (tree) at (7.2, 1.5) {$\Psi$};
	\node[detnode] (varv) at (3.6+0.9, 0) {\tiny $\sigma^2_{g_{v}}(\bar{t})$};
	\node[constnode] (tbar) at (3.6+0.9+1.8, 0) {$\bar{t}$};
	\node[snode] (v) at (3.6, 3.25) {$\vec{p}$};
	\draw[taro] (theta) -- (v);
	\draw[taro] (g0) -- (v);
	\draw[taro] (alpha) -- (v);
	\draw[taro] (sigma) -- (v);
	\draw[taro] (tree) -- (v);
	\draw[taro] (varv) -- (sigma);
	\draw[taro] (tbar) -- (sigma);
	\draw[taro] (tbar) -- (tree);
	\draw[taro] (alpha) -- (sigma);
	\node[white] (varpic) at (3.6+0.9, -1.75) {};
	\draw[taro] (varpic) -- (varv);
	\node (vdist) at ($(v) + (1.8, 0.3)$) {Gaussian};
	\draw (v) -- (vdist);
	\draw[decoration={brace, mirror, raise=18pt}, sloped, decorate] (7.2, 1.5 - .5) -- node[below = 18pt] {OU parameters} (7.2, 1.5 + .5);
	\end{scope}
	
	% Host 
	\begin{scope}[xshift = 0cm, yshift = -7.5cm]
	\node[anchor = west] (clabel) at (-1, 5.55) {\textbf{C} Generating host effects};
	\node[constnode] (d) at (1.8, 1.5) {2};
	\node[detnode] (p) at (3.6, 1.5) {$g$};
	\node[snode] (G) at (1.8, 3) {$G$};
	\node[rectangle, dashed, very thick, inner sep=4mm, draw=black!100, fit = (G) ] (Gplate) {};
	\node[anchor=north west, inner sep=3pt] at (Gplate.north west) {\tiny $N\times M$};
	\node[detnode] (varh) at (3.6, 0) {$\sigma^2_{h}$};
	\node[constnode] (delta) at (3.6, 3) {$\delta$};
	\node[detnode] (h) at (3.6, 4.5) {$\vec{h}$};
	\node[constnode] (M) at (5.4, 3) {$M$};
	\draw[taro] (d) -- (G);
	\draw[taro] (p) -- (G);
	\draw[taro] (varh) -- (p);
	\draw[taro] (delta) -- (p);
	\draw[taro] (M) -- (p);
	\draw[taro] (M) -- (h);
	\draw[taro] (delta) -- (h);
	\draw[taro] (G) -- (h);
	\node[white] (varpic) at (3.6, -1.5) {};
	\draw[taro] (varpic) -- (varh);
	\node (hdist) at ($(h) + (2.7, 0.3)$) {$\approx$ Gaussian};
	\draw (h) -- (hdist);
	\node (gdist) at ($(G) + (-2, 0.6)$) {Binomial};
	\draw (G) -- (gdist);
	\end{scope}
	
	% Environment
	\begin{scope}[xshift = 9cm, yshift = -6.5cm]
	\node[anchor = west] (dlabel) at (-1, 3) {\textbf{D} Generating environmental effects};
	\node[detnode] (vare) at (0, 0) {$\sigma^2_{\epsilon}$};
	\node[constnode] (zero) at ($(vare) + (1.8, 0)$) {0};
	\node[snode] (e) at ($(vare) + (1.8/2, 1.5)$) {$\vec{\epsilon}$};
	\draw[taro] (zero) -- (e);
	\draw[taro] (vare) -- (e);
	\node[white] (varpic) at (0, -1.5) {};
	\draw[taro] (varpic) -- (vare);
	\node (edist) at ($(e) + (1.8, 0.3)$) {Gaussian};
	\draw (e) -- (edist);
	\end{scope}
	
	\end{tikzpicture}
	\caption{A graphical model representation of our simulation scheme, following the recommendations in \cite{Hohna2014}. Variables in solid squares are constants, with the two master control variables that we vary from simulation to simulation highlighted in red. Variables in solid circles are realizations of random variables and variables in dashed circles are determined as a function of other variables. Arrows represent dependencies among variables and the dashed square represents repetition. All parameters are defined in Table \ref{tab:sim-params}, as well as the values or expressions used for them. (A) shows how the variance in the simulated environmental effect $\sigma^2_{\epsilon}$ is smaller if the master pathogen heritability value $H^2_{\bar{t}}$ is higher and vice-versa. (B) shows the OU parameters and the pathogen phylogeny, which generate the Gaussian-distributed pathogen effects. The OU parameters $\theta$ and $g_0$ are fixed, whereas $\sigma$ is a deterministic function of the variance in the pathogen effect and the value of $\alpha$. In other words, we use $\sigma$ to maintain the desired pathogen heritability while varying $\alpha$. We generate a new random phylogeny for each simulation. (C) shows how host genotypes are drawn to generate host effects. The host genotype matrix $G$ contains the number of copies (0, 1, or 2) for each of $M$ causal variants with effect size $\delta$. We assume half the variants have a positive effect and half have a negative effect. The allele frequency $g$ for the causal variants set so that we achieve the desired variance in the host effects. (D) shows that the environmental effect is drawn from a Gaussian distribution with mean zero and variance as determined in part (A).}
	\label{fig:sim-design}
\end{figure}

\begin{figure}[H]
\begin{center}
\includegraphics[width = 0.7\linewidth]{figures/sigma_vs_alpha.png}
	\caption{Relationship between the selection strength parameter $\alpha$ and the intensity of stochastic evolutionary fluctuations parameter $\sigma$ at two different heritability ($H^2$) values in the  simulation scheme. $\sigma$ was determined as a function of $\alpha$ and $H^2$ under the POUMM (function given in Table \ref{tab:sim-params}).}
	\label{fig:sigma-vs-alpha}
	\end{center}
\end{figure}

\begin{figure}[H]
\begin{center}
\includegraphics[width = 0.7\linewidth]{figures/spvl_calculation_comparison.png}
	\caption{A comparison of different ways to calculate spVL based on viral load measurements provided by the SHCS. The stricter filtering excludes all measurements possibly $<$ 6 months after infection and after treatment or AIDS, whereas the more lenient filtering excludes only measurements after treatment. We used the lenient filter, mean measurement values because these correlate well with the values from the stricter filter but allow us to retain many more individuals from the cohort for our study.}
	\label{fig:spvl-calc-comparison}
	\end{center}
\end{figure}

\begin{figure}[H]
\begin{center}
\includegraphics[width = \linewidth]{figures/spvl_on_tree.png}
	\caption{Inferred HIV-1 $pol$ gene phylogeny with tips colored by (A) calculated spVL, (B) estimated non-pathogen effects on spVL and (C) estimated pathogen effects on spVL.}
	\label{fig:spvl-on-tree}
	\end{center}
\end{figure}

\begin{figure}[H]
\begin{center}
\includegraphics[width = 0.462\linewidth]{figures/trait_values.png}
	\caption{A comparison of measured (calculated) spVL values versus our estimated non-pathogen effect on spVL for each SHCS cohort member used in the study. The histograms show the marginal distribution of each value across the individuals.}
	\label{fig:trait-comparison}
	\end{center}
\end{figure}

\begin{figure}[H]
	\centering
		\includegraphics[width=0.7\linewidth]{figures/poumm_parameter_estimates.png}
		\caption{Posterior distributions compared to the prior for POUMM parameter estimates based on SHCS data. We ran two different MCMC chains to ensure the estimates converged.}
\label{fig:poumm-parameters}
\end{figure}

\begin{figure}[H]
	\centering
	\includegraphics[width=\linewidth]{figures/qq_plots.png}
	\caption{Quartile-quartile plots from association tests. The dashed green line shows the y = x line.}
	\label{fig:qq-plots}
\end{figure}

\begin{figure}[H]
	\includegraphics[width=\linewidth]{figures/host_genotype_pca.png}
	\caption{SHCS individuals and HapMap3 individuals plotted along the top three principle components of genetic variation. Points with black borders are within the thresholds used to select individuals of likely European ancestry.}
	\label{fig:PCA}
\end{figure}

\begin{figure}[H]
\begin{center}
\includegraphics[width = \linewidth]{figures/qdr_on_tree_arboricola.png}
	\caption{Inferred \emph{X. arboricola} phylogeny with tips colored by (A) calculated QDR, (B) estimated non-pathogen effects on QDR and (C) estimated pathogen effects on QDR.}
	\label{fig:qdr-on-tree}
	\end{center}
\end{figure}

\begin{figure}[H]
	\centering
		\includegraphics[width=0.7\linewidth]{figures/poumm_parameter_estimates_arboricola.png}
		\caption{Posterior distributions compared to the prior for POUMM parameter estimates based on \emph{A. thaliana}-\emph{X. arboricola} data. We ran two different MCMC chains to ensure the estimates converged.}
\label{fig:poumm-parameters-qdr}
\end{figure}

\begin{figure}[H]
\begin{center}
\includegraphics[width = 0.462\linewidth]{figures/trait_values_arboricola.png}
	\caption{A comparison of measured (calculated) mean QDR values across all \emph{A. thaliana} accession pairings and replicates  versus our estimated non-pathogen effect on mean QDR for each \emph{X. arboricola} pathogen strain. The histograms show the marginal distribution of each value across the strains. The Pearson correlation coefficient is 1.}
	\label{fig:trait-comparison-qdr}
	\end{center}
\end{figure}

\end{doublespace}  % don't doublespace tables

\newpage

\begin{table}[H]
\caption{POUMM parameter estimates for spVL based on SHCS data. HPD = Highest posterior density.}
	\begin{tabularx}{\linewidth}{XXX} \hline 
	Parameter & Posterior mean & 95\% HPD \\ \hline 
	${g_{0}}$ & 4.23 & (1.72, 6.71) \\
	$\theta$ & 4.47 & (4.37, 4.58) \\
	$\sigma$ & 5.25 & (2.37, 7.9) \\
	$\alpha$ & 57.65 & (19.49, 95.2) \\
	$\sigma_e$ & 0.54 & (0.43, 0.65) \\ 
	$H^2_{\bar{t}}$ & 0.45 & (0.24, 0.67) \\ \hline
	\end{tabularx}
	\label{tab:POUMM-parameters-our-estimates}
\end{table}

\begin{table}[H]
\caption{POUMM parameter estimates for QDR based on \emph{A. thaliana}-\emph{X. arboricola} data data. HPD = Highest posterior density.}
	\begin{tabularx}{\linewidth}{XXX} \hline 
	Parameter & Posterior mean & 95\% HPD \\ \hline 
	${g_{0}}$ & 0.40 & (0.01, 0.78) \\
	$\theta$ & 0.39 & (0.30, 0.49) \\
	$\sigma$ & 0.03 & (0.0, 0.06) \\
	$\alpha$ & 0.03 & (0.0, 0.05) \\
	$\sigma_e$ & 0.18 & (0.08, 0.27) \\ 
	$H^2_{\bar{t}}$ & 0.33 & (0.0, 0.77) \\ \hline
	\end{tabularx}
	\label{tab:POUMM-parameters-our-estimates-qdr}
\end{table}

% latex table generated in R 4.1.1 by xtable 1.8-4 package
% Sat Oct 30 10:31:16 2021
\begin{table}[ht]
\caption{Effect size and p-values from the top most strongly associated variants in the $\emph{CCR5}$ and MCR regions from each of the two GWAS performed in our study. ``Standard'' means the GWAS with standard spVL trait values and ``Corrected'' means the GWAS with the estimated non-pathogen part of the trait. Entries above the dividing line are the top-associated variants from the ``Standard'' GWAS and entries below the dividing line are the top-associated variants from the ``Corrected'' GWAS. Many entries overlap between the two.}
\begin{tabularx}{\linewidth}{XXXXXXX}
  \hline
    Region & Position & Variant & Standard effect size & Standard p-value & Corrected effect size & Corrected p-value \\ 
  \hline
    $\emph{CCR5}$ & 46531144 & rs9845968 & -0.16 & $5.6 \times 10^{-9}$ & -0.083 & $1.2 \times 10^{-7}$ \\ 
    $\emph{CCR5}$ & 46537849 & rs867620 & -0.16 & $3.2 \times 10^{-9}$ & -0.085 & $6 \times 10^{-8}$ \\ 
    $\emph{CCR5}$ & 46539864 & rs11130092 & -0.16 & $1.1 \times 10^{-9}$ & -0.087 & $2.6 \times 10^{-8}$ \\ 
    $\emph{CCR5}$ & 46540932 & rs10865942 & -0.16 & $8.4 \times 10^{-9}$ & -0.081 & $4 \times 10^{-7}$ \\ 
    $\emph{CCR5}$ & 46541147 & rs7430431 & -0.17 & $9.2 \times 10^{-10}$ & -0.088 & $2.3 \times 10^{-8}$ \\ 
    MHC & 31274380 & rs9264942 & -0.21 & $4.5 \times 10^{-13}$ & -0.12 & $3.7 \times 10^{-13}$ \\ 
    MHC & 31321919 & rs1055821 & -0.33 & $9.4 \times 10^{-13}$ & -0.19 & $1.4 \times 10^{-12}$ \\ 
    MHC & 31380034 & rs112243036 & -0.32 & $9.9 \times 10^{-16}$ & -0.17 & $3.7 \times 10^{-14}$ \\ 
    MHC & 31391401 & rs4418214 & -0.34 & $2.4 \times 10^{-14}$ & -0.18 & $2.5 \times 10^{-12}$ \\ 
    MHC & 31400137 & rs138130755 & -0.46 & $1 \times 10^{-12}$ & -0.26 & $2.6 \times 10^{-12}$ \\ 
    MHC & 31400705 & rs138117378 & -0.46 & $1 \times 10^{-12}$ & -0.26 & $2.6 \times 10^{-12}$ \\ 
    MHC & 31402358 & rs148792134 & -0.46 & $1 \times 10^{-12}$ & -0.26 & $2.6 \times 10^{-12}$ \\ 
    MHC & 31409677 & rs140991764 & -0.46 & $1 \times 10^{-12}$ & -0.26 & $2.6 \times 10^{-12}$ \\ 
    \hline
    $\emph{CCR5}$ & 46531144 & rs9845968 & -0.16 & $5.6 \times 10^{-9}$ & -0.083 & $1.2 \times 10^{-7}$ \\ 
    $\emph{CCR5}$ & 46537849 & rs867620 & -0.16 & $3.2 \times 10^{-9}$ & -0.085 & $6 \times 10^{-8}$ \\ 
    $\emph{CCR5}$ & 46539864 & rs11130092 & -0.16 & $1.1 \times 10^{-9}$ & -0.087 & $2.6 \times 10^{-8}$ \\ 
    $\emph{CCR5}$ & 46541147 & rs7430431 & -0.17 & $9.2 \times 10^{-10}$ & -0.088 & $2.3 \times 10^{-8}$ \\ 
    $\emph{CCR5}$ & 46556835 & rs6808142 & 0.15 & $8.3 \times 10^{-8}$ & 0.082 & $3.3 \times 10^{-7}$ \\ 
    MHC & 31274380 & rs9264942 & -0.21 & $4.5 \times 10^{-13}$ & -0.12 & $3.7 \times 10^{-13}$ \\ 
    MHC & 31321919 & rs1055821 & -0.33 & $9.4 \times 10^{-13}$ & -0.19 & $1.4 \times 10^{-12}$ \\ 
    MHC & 31367874 & rs111281598 & -0.37 & $1.5 \times 10^{-12}$ & -0.22 & $2.1 \times 10^{-12}$ \\ 
    MHC & 31376266 & rs73400361 & -0.37 & $1.4 \times 10^{-12}$ & -0.22 & $2.1 \times 10^{-12}$ \\ 
    MHC & 31380034 & rs112243036 & -0.32 & $9.9 \times 10^{-16}$ & -0.17 & $3.7 \times 10^{-14}$ \\ 
   \hline
\end{tabularx}
\label{tab:gwas-top-associations}
\end{table}

\begin{table}[H]
\caption{Summary statistics for log spVL in previously sampled populations. $\bar{z}$ is average spVL (log copies/mL) and $\sigma^2_z$ is variance in measured spVL ( log $\textrm{copies}^2/\textrm{mL}^2$). Values from \protect\citep{Blanquart2017, Mitov2018} are empirical; values from \protect\citep{Bonhoeffer2015} were estimated by fitting a normal distribution to the data.}
	\begin{tabularx}{\linewidth}{XXX} \hline 
	Measurement & Value & Reference \\ \hline 
	$\bar{z}$ & $\approx$ 4.5 & \citet{Mitov2018} \\
	$\bar{z}$ &  4.4 & \citet{Blanquart2017} \\
	$\bar{z}$ & $\approx$ 4.5 & \citet{Bonhoeffer2015} \\
	$\sigma^2_z$ &  0.73 &  \citet{Mitov2018} \\
	$\sigma^2_z$ &  0.50 & \citet{Blanquart2017} \\
	$\sigma^2_z$ & $\approx$ 0.5 & \citet{Bonhoeffer2015} \\ \hline 
	\end{tabularx}
	\label{tab:spVLmeasurements}
\end{table}
%\todo{I guessed at bar(z) from figure in Mitov2018, search supplement for exact}

\begin{table}[H]
\caption{POUMM parameter estimates for spVL from previous studies.}
	\begin{tabularx}{\linewidth}{llXXX}
	\hline 
	Parameter & Value (Uncertainty) & Reference & Notes \\
	\hline 
	$g_0$ & 5.54 (4.04 - 7.25) & \citet{Mitov2018} & 8,483 UK HIV cohort individuals, $pol$ tree \\
	$\theta$ & 4.45 (4.41 - 4.49) & \citet{Mitov2018} & \\
	$\theta$ &  4.0 (1.6 - 4.)) & \citet{Bertels2018} & 3,036 SHCS individuals, $pol$ tree \\
	$\theta$ & 4.1 (3.5 - 4.9) & \citet{Blanquart2017} & 1,581 subtype B individuals from Europe, whole genome tree \\
	$\alpha$ & 28.78 (16.64 - 46.93) & \citet{Mitov2018} & \\
	$\alpha$ & 32.7 (0.03 - 57.6) & \citet{Bertels2018} & \\
	$\alpha$ & 7.6 (1.2 - 10) & \citet{Blanquart2017} & **limited $\alpha$ to $\le$ 10 \\
	$\sigma$ & 2.97 (1.95 - 4.37) & \citet{Mitov2018} & \\
	$\sigma$ & 1.3 (0.66 -1.87) &  \citet{Blanquart2017} & \\
	$\sigma_e$ & 0.77 (0.73, 0.8) & \citet{Mitov2018} & \\
	$\sigma_e$ & 0.61 (0.54, 0.65) &  \citet{Blanquart2017} & \\
	\hline 
	\end{tabularx}
	\label{tab:POUMMparams-prev-estimated}
\end{table}

\begin{table}[H]
\caption{Number of samples for GWAS after sequential filtering steps.}
    \begin{tabularx}{\linewidth}{XX}
    \hline
    Sample filter & Number of samples remaining \\
    \hline
    Subtype B pol sequences & 1516 \\
    With paired spVL measurement & 1516 \\
    $>$ 750 characters in sequence & 1493 \\
    Individual is of European ancestry & 1396 \\
    Kinship coefficient $>$ 0.09375 & 1392 \\ 
    \hline
    \end{tabularx}
    \label{tab:sample-filtering}
\end{table}

\begin{table}[H]
\caption{Number of variants for GWAS after sequential filtering steps.}
	\begin{tabularx}{\linewidth}{XX}
	\hline
	Variant filter & Number of variants remaining \\
	\hline 
	Raw data & 76979521 \\
	Missing genotype rate $>$ 0.05 & 11590002 \\
	Hardy-Weinburg exact test p-value $< 5 \times 10^{-5}$ & 11589246 \\
	Minor allele frequency $<$ 0.01 & 6228626 \\ \hline			
	\end{tabularx}
	\label{tab:variant-filtering}
\end{table}

\stopsection
\newpage
\section*{Editors’ comments}
AE: I have now received two reviews of your paper. Both reviewers were excited about the methodological concept, but both where also concerned about 1) the perceived narrow application of the methodology and 2) the example based on data which are not accessible. The reviewers have suggested ways to perhaps broaden the potential applications of the method and have even suggested additional (alternative) data sets that ARE accessible.  I strongly encourage you to utilize data that others can access to explore your methods. If you want to make insights on HIV evolution using restricted data, then please send such work to a medical journal. But for MBE, the expectation (especially for a methodological paper) is that the demonstrations of methods should be using data that are easily and openly accessible.  Hopefully, you will find these reviews helpful in revising your work.

\textcolor{blue}{We thank the editor for the chance to improve our manuscript and the two reviewers for their constructive feedback. We feel the quality of the manuscript improved thanks to their suggestions. Below, we highlight the main improvements in the revised manuscript. We also discuss two suggestions we did not implement. Finally, in the point-by-point responses, we highlight the specific changes to the manuscript based on each of the reviewers comments.}

\textcolor{blue}{The main improvement in the revised manuscript is that we added an analysis of an open dataset and re-phrased our abstract and introduction to focus on the broader applicability of our method, including what differentiates it from previous methods. We highlight that our two-step method provides additional insights on the evolutionary dynamics of a trait in a pathogen population and that it is compatible with a variety of downstream GWAS association testing methods. We removed the HIV-specific information from the introduction and moved the relevant comparisons to the discussion section.}

\textcolor{blue}{The only suggestions we did not implement were to re-run the HIV GWAS using a linear mixed model and to perform the HIV analysis again using elite controllers. In the first case, we rephrased the discussion to make it clearer that our method is envisioned as a pre-processing step that can be used prior to GWAS association testing under any model. Given the revised focus of the manuscript away from HIV-1 specific insights, we felt improving the HIV-1 GWAS association testing methodology was beyond the scope of the current manuscript. In the second case, our dataset does not contain enough elite controllers to perform the additional analysis.}

\section*{Reviewer 1}

The authors present a new GWAS method that estimates and removes trait variability due to the pathogen using information from the full pathogen phylogeny. I do not have critiques about the framework, as the method the authors have developed seems novel and in principal, groundbreaking. However, I am finding difficult to grasp the usability of it outside of pathogens that integrates in host genome. What is the broad applicability of the method? Is this method only applicable to stVL in the context of HIV? What else could be investigated? Is it possible to pair transcriptomic data? Can author expand on this?

\vspace{5mm}
\textcolor{blue}{We edited the introduction to make it clearer when this method is applicable: ``This framework relies on paired pathogen-host genotyping and is envisioned specifically for continuous-valued traits that are highly heritable from infection partner to infection partner.''}

\textcolor{blue}{We also expanded the discussion to include the possibility of transcriptomic applications and highlight that even when this method is not anticipated to increase GWAS power, it is still interesting to fit the POUMM and estimate pathogen effects on different traits: ``Future applications of our method might investigate other clinically significant disease traits and outcomes that are affected by both host and pathogen genetic factors, for instance Hepatitis B Virus-related hepatocellular carcinoma \citep{An2018HostCarcinoma}, Hepatitis C treatment success \citep{Ansari2017Genome-to-genomeVirus}, and susceptibility to or severity of certain bacterial infections, e.g. \cite{Messina2016ImpactInfections, Donnenberg2015BacterialStudies}. Transcriptomic data has also previously been modeled as an evolving phenotype using an Ornstein-Uhlenbeck model \citep{Rohlfs2014ModelingVariation}. Thus, one could also estimate pathogen effects on host gene expression. In particular, it will be of interest to understand the evolutionary dynamics of these traits in the respective pathogen populations.''}

\begin{enumerate}
    \item It would have been interesting to see how their methods performs with another type of pathogen, a virus that does not integrate, such SARS-CoV-2, or a bacterial one, such TB. \\
    \textcolor{blue}{We added an analysis of \emph{A. thaliana} quantitative disease resistance to the phytopathogenic bacteria \emph{X. arboricola}. The findings are highlighted in the discussion: ``For \emph{A. thaliana} QDR, inferred phylogenetic correlations are too weak to justify a phylogenetic correction (maximum expected correlation between strains was $3.2 \times 10^{-12}$). \cite{Wang2018Two-wayGenomes} found no significant \emph{A. thaliana} variants associated with QDR using a linear mixed model jointly accounting for host genetic effects, pathogen genetic effects, and interaction effects. As with HIV-1 spVL, our results do not challenge this previous finding.''}
    \item In the case of co-infection, as sometimes is the case of HIV and TB, how would the method perform? \\
    \textcolor{blue}{We addressed this point in the discussion: ``Finally, additional model complexity can be added to the GWAS association tests. For instance, our method does not account for co-infection, which might add additional variance to trait values and decrease GWAS power. In this case, one could add co-infection status as a covariate in the GWAS association test to account for this variable.''}
    \item In the case of HIV, is it possible to compare the data with elite controllers? \\
    \textcolor{blue}{Unfortunately our dataset does not include a sufficient number of elite controllers to perform this comparison. The SHCS cohort includes 83 individuals classified as elite controllers, as defined by at least three measurements of plasma virus load (VL) $<$ 2000 RNA copies/ml over at least a 12-month period in the absence of antiviral therapy (Pereya et al, Science, 2010). spVL values range from 1.3 to 3.1 in this group (all very low). After filtering steps, only 8 of these individuals were retained for the GWAS, so the sample size is unfortunately not sufficient to do a comparative GWAS. However, we note that McLaren et al. 2015 found similar signal of association on chromosome 3 and 6 when they filtered to elite controllers in a standard GWAS setup.}
    \item In the case of determining whether viral effects may be confounding or extraneous variables that bias estimates of host genetic effects, I am not sure I understand the premise of this hypothesis. Can authors elaborate on it more in detail? \\
    \textcolor{blue}{We edited the introduction to make it clearer why one should consider pathogen effects in a host GWAS: ``Shared ancestry among individuals, especially between close relatives, can give rise to spurious genetic correlations with a trait. Corrections for these types of population structure in human GWAS cohorts are well-developed and widely accepted  \citep{Astle2009, Price2006}. More recently, analogous methods have been developed for microbial GWAS, where clonal reproduction exacerbates population structure \citep{Power2017}... [our method] estimates and removes pathogen effects on the trait so that downstream GWAS association testing with host genetic variants can be performed as normal under the assumption of independent samples''}
    \item What type of clinical information/trait is included under the environmental effect? How the diversity in environmental effects across the population affects the method? \\
    \textcolor{blue}{We edited the text to give examples of factors that would be grouped into the environmental effect under the POUMM model we use: ``Variation in infectious disease traits like viral load or infection severity can come from several sources. These include host genetic factors, pathogen genetic factors, interaction effects between the host and the pathogen, or non-genetic factors like healthcare quality or temperature... In the infectious disease GWAS case, we assume the heritable part comprises pathogen genetic factors and all other factors are non-heritable.''}
    
    \textcolor{blue}{In addition to the change described in response 1, we also added text to the new approaches section to make it clearer that our method is complementary to but does not replace other methods to account for diversity in environmental effects (we only account for heritable pathogen genetic effects): ``GWAS typically stratify samples or include covariates to correct for host genetic factors or non-genetic factors that may be correlated with a trait value. This leaves pathogen genetic factors as a remaining source of correlation, since close transmission partners may be infected with very similar pathogen strains.''}
    \item Lines 194-196:  please expand on the parameters of the simulation in greater detail. \\
    \textcolor{blue}{We edited this paragraph to provide a full description of the simulation scheme.}  
    \item Lines 205-207: it would be best, if there is not a limitation in the number of words, to report here “why” instead of saying that it has been shown in SI. \\
    \textcolor{blue}{We added the suggested explanation: ``As shown in the supplemental material, we can calculate the expected RMSE using the scaled trait value across scenarios in our simulation scheme because the variance in the trait due to viral genetic effects and environmental effects is fixed. Thus, we expect the RMSE using the scaled trait value to be 0.74 across all simulation scenarios.''}
    \item Liens 212: “our method performs well when an infectious disease trait is highly heritable”, I found the concept of heritability in the infectious disease filed confusing. Are the authors talking about traits in the viral population? Is it possible to expand the sentence with more details for clarity? \\
    \textcolor{blue}{We edited this statement to be clear that we mean ``pathogen heritability''. We also tried to explain the concept of heritability due to the pathogen better in the introduction: ``Variation in infectious disease traits like viral load or infection severity can come from several sources. These include host genetic factors, pathogen genetic factors, interaction effects between the host and the pathogen, or non-genetic factors like healthcare quality or temperature. GWAS typically stratify samples or include covariates to correct for host genetic factors or non-genetic factors that may be correlated with a trait value. This leaves pathogen genetic factors as a remaining source of correlation, since close transmission partners may be infected with very similar pathogen strains. Broad-sense pathogen heritability $H^2$ quantifies the fraction of total variance in a trait that is ``inherited'' from infection partner to infection partner, i.e., due to pathogen factors.''}
\end{enumerate}

\section*{Reviewer 2}

In this paper, Nadeau and colleagues look to correct for pathogen heritability in studies of host association with infectious diseases. They do this by extending an existing method which models trait evolution as a walk along a phylogeny, and provide a new method which which decomposes a pathogen trait into pathogen heritability vs 'other effects' using maximum-likelihood. By using only the component corresponding to 'other effects' as the phenotype in a standard host GWAS, they are able to correct for dependence between pathogen samples due to shared ancestry. As the authors note, this method is rather convenient as it results in an adjusted phenotype which can be used with standard GWAS tools. They apply their method to simulated data, showing a range (high heritability and selection) where a power advantage should be gained. They also apply the method to HIV set point viral load (spVL).

The paper is based on a sound idea, and is a technically competent solution, with no major scientific issues I could identify. Code is provided; data available via managed access; derivations in methods were appropriately explained. The introduction and conclusion are both well-written and do not overstate the results.

The main comment I have is based perhaps more on the impact of the study. The study is very strongly focused on analysis of HIV-1 spVL, which is not really the ideal trait for this method (as seen from the simulations in fig 2; and fig S4 showing the trait changing not all that much). It has long been established that there are two clear host associations, which indeed are exactly what the authors find before and after their adjustment. [If the aim was to try and find more associations for this trait, it would likely be much better to collect lots more samples than change the association methods, as has been found in host GWAS studies of other traits]. To increase the relevance of this study, the authors may consider whether they can apply their approach to other pathogens/traits. At the same time, I also appreciate that host/pathogen data is hard to come by and may have access issues that are too difficult to overcome. However, I do think if any of these studies in other pathogens could also be analysed here, it would really add to the paper. The authors cite a study of HCV as well as some other viral studies as examples, and I would also point to two more in bacteria (https://www.pnas.org/content/115/24/E5440; https://www.nature.com/articles/s41467-019-09976-3), and one in malaria (https://www.biorxiv.org/content/10.1101/2021.03.30.437659v1). Perhaps similar data is now available for SARS-CoV-2 also?

\textcolor{blue}{We were able to access additional data from the \emph{A. thaliana}-\emph{X. arboricola} pathosystem to demonstrate the broader applicability of our method. Unfortunately neither dataset we analyzed had sufficiently highly correlated trait values amongst related pathogen strains to generate novel biological insights. We edited the text to emphasize that an advantage of our approach is that one gets information on the evolutionary dynamics of the trait in  the pathogen population, even if GWAS results are not changed.}

\vspace{5mm}
I also have some more minor comments:

\begin{enumerate}
    \item How does the method compare to the ATOMM approach in https://www.pnas.org/content/115/24/E5440? Can this be applied to any of the simulated or real data here for comparison. \\
    \textcolor{blue}{We cannot apply ATOMM directly to our simulated data because we do not simulate pathogen genotypes, rather we simulate only pathogen trait values. Further, ATOMM is only implemented for haploid alleles, making it not ideal for the HIV data analyzed. Instead, we were able to access the same dataset ATOMM was tested on in their original publication. This allows us to compare GWAS results from our method versus the ATOMM method, which accounts for the pathogen and host genome jointly in the same linear mixed model and allows tests for marginal effects of host and pathogen alleles as well as pairwise host allele $\times$ pathogen allele interaction effects.}
    
    \textcolor{blue}{We present the findings of this comparison in the discussion: ``For \emph{A. thaliana} QDR, inferred phylogenetic correlations are too weak to justify a phylogenetic correction (maximum expected correlation between strains was $3.2 \times 10^{-12}$). \cite{Wang2018Two-wayGenomes} found no significant \emph{A. thaliana} variants associated with QDR using a linear mixed model jointly accounting for host genetic effects, pathogen genetic effects, and interaction effects. As with HIV-1 spVL, our results do not challenge this previous finding.''}
    
    \textcolor{blue}{We also highlight the linear mixed model approach presented in ATOMM as an alternate means to take pathogen effects into accout: ``As in the \emph{A. thaliana}-\emph{X. arboricola} application, fitting the POUMM may reveal that expected phylogenetic correlations between samples are not strong enough to justify using the corrected trait values in a GWAS. In this case, one may wish to use a linear mixed model as in \citep{Wang2018Two-wayGenomes}, where the pathogen effect is co-estimated as a random effect.''}
    \item As I understand it the POUMM method assumes a 'correct' phylogeny. Do the authors have any thoughts about including uncertainty in the phylogeny (for example by using a posterior sample from e.g. mrbayes) and whether this may affect their approach? Relatedly, does a tree from just pol sequences (L342) give enough resolution? \\
    \textcolor{blue}{TODO: improve phylogeny by masking sites under selection. We randomly chose an alternate tree from the collection of optimal trees produced with the IQ-TREE ``wt'' option and re-fit the POUMM. Posterior mean parameter estimates were comparable. TODO: re-correct trait values and re-run the GWAS using a few different trees.}
    
    \item It would be useful to include more histograms and scatterplots of traits before/after the adjustment, as in figure S4. In particular, doing this plot again for one of the simulations where it does improve power would be informative. In my opinion, these could be a main figure. \\
    \textcolor{blue}{We implemented this suggestion and added two new main text figures. They are discussed in the section ``Estimator accuracy'': ``Figure \ref{fig:simulationTraitExample} gives some intuition for how this correction works by contrasting simulated scenarios with high and low heritability and low selection strength/ low stochastic fluctuations. These parameter combinations mean trait values are phylogentically correlated at clustered tips in the phylogeny. This information is the basis from which our method estimates the pathogen and corresponding non-pathogen effects on the trait. Figure \ref{fig:tip-trait-correlation} shows in more detail how trait values are correlated at closely clustered tips when an infectious disease trait is highly heritable and selection strength is low/ stochastic fluctuations are low.''}
    \item Do the authors know how the computational requirements of their ML estimation method scales with sample size? Would it work for a large SARS-CoV-2 phylogeny for instance? \\
    \textcolor{blue}{The POUMM R package for estimating the POUMM model parameters runs in less than an hour on a tree of 10,000 tips and has been shown to scale to trees of up to 100,000 tips. We additionally verified that the matrix inversion steps required for the last ML estimation step based on the fitted POUMM parameters ran successfully on a simulated tree of 10,000 tips using 4 cores with maximum memory use of 6500MB per core. We now highlight the method's scalability in the discussion: ``The method relies on the freely available R package POUMM \citep{Mitov2017a-POUMM}, which scales to trees of up to 10,000 tips \citep{Mitov2019}.''}
    \item Fig S3 would be useful, but isn't really readable at present. Perhaps consider an interactive tree viewer to make this more useful. \\
    \textcolor{blue}{We added the same figure for the \emph{A. thaliana}-\emph{X. arboricola} dataset, which is much more legible due to having fewer samples (Figure \ref{fig:qdr-on-tree}). We also added sub-sampled phylogenies from the simulation study annotated with trait values in Figure \ref{fig:simulationTraitExample} to demonstrate what phylogenetically-correlated trait values look like. Finally, we now reference the highest correlation values expected between any two tips in the HIV-1 and \emph{X. arboricola} phylogenies to add a quantative element to the discussion of phylogenetic correlations: ``The highest expected correlation in trait values between any two tips in the HIV-1 phylogeny under the POUMM was 0.45. However, Figure \ref{fig:spvl-on-tree} shows that this trait is not obviously phylogenetically structured in the cohort in general, despite high heritability... Given the posterior mean estimates for the POUMM parameters [using \emph{X. arboricola} data], expected correlation in trait values between tips were very low (maximum value $3.2 \times 10^{-12}$ compared to maximum value of 0.45 in the HIV-1 spVL application). Thus, the phylogeny is not very informative for a trait value correction.''}
    \item The authors I think only look at power/TPR. What about the effect on false positive rate? \\
    \textcolor{blue}{We re-ran the simulations to assess the false positive rate and include the results in the Materials and methods section: ``For our main results (Figure \ref{fig:simulationResults}) we simulated 20 truly associated variants, as described above. To also check the false positive rate (FPR), we re-ran the simulations with an additional 80 non-associated variants. Across all the association tests in this second simulation setup (7 $H^2$ levels $\times$ 10 $\alpha$ levels $\times$ 100 variants $\times$ 20 replicates per scenario = 140,000 association tests), FPR was 0.0005 using the true (simulated) non-pathogen part of the trait, 0.0005  using the estimated non-pathogen part of the trait, and 0.0006 using the scaled total trait value. These rates are comparable to the expected FPR of 0.0005 at significance level 0.05 corrected for 100 tests. Given the stricter correction for multiple testing in this second simulation setup, the TPR decreased significantly across all three GWAS response variables used.''}
    \item I don't think panel A of figure 2 is explained well by the caption or text, I could not understand what the RMSE is showing here. What does 'more accurate' look like in this plot? \\
    \textcolor{blue}{We edited the figure legend to make it clearer: ``(A) shows the root mean squared error (RMSE) of our estimator (left) compared to un-corrected trait values, scaled by their mean (right) under each simulated evolutionary scenario. The RMSE is with reference to the true (simulated) host part of the trait values. Thus, more accurate estimates (lower RMSE) mean the trait value used for GWAS will be closer to the true host part of the trait value.''}
    \item Is there any explanation why the simulated (actual) and estimated non-pathogen trait in fig 2B start to look quite different at stronger selection strengths. \\
    \textcolor{blue}{We added a figure (Figure \ref{fig:tip-trait-correlation}) to illustrate how the correlation between clustered tips on the phylogenetic tree decreases with increasing selection strength/ increasing stochasticity. Without phylogenetic signal, our method has no information to estimate the pathogen effect on the trait.}
    \item Using plink and removing samples with divergent ancestry is probably a little outdated for host GWAS. A linear mixed model would likely provide more power still. In particular, I'd suggest warped-lmm (http://dx.doi.org/10.1038/ncomms5890), which also transforms the phenotype to increase power. Furthermore, I would be interested to see histograms/scatter plots of a few phenotypes: a) no adjustment; b) warped-lmm only; c) the author's method; d) both adjustments. Perhaps the methods are similar, or using both does increases power beyond either method alone. \\
    \textcolor{blue}{We highlight that our method is a pre-processing step before GWAS and can be combined with many different GWAS models implemented in softwares like PLINK, GEMMA, and warped-lmm. Our revised manuscript focuses more on the method rather than generating biological insights for HIV or \emph{X. arboricola}, so we feel that updating our example GWAS is beyond the scope of the current study. We added this explanation to the discussion: ``Our approach is envisioned to be complementary to previously developed methods for infectious disease GWAS... In cases where a correction can be estimated and applied, it is still possible to use any of the number of previously developed GWAS models for the actual association testing (we used a linear model approach implemented in PLINK \citep{Chang2015}, though a more advanced method would be to use a linear mixed model with host ancestry as a random effect).''}
\end{enumerate}

\vspace{5mm}
Suggested text changes:

\begin{enumerate}
    \item The first sentence of the abstract is probably also true for many common diseases analysed in human GWAS (apart from perhaps the pathogen, but gut microbiome may be a factor too). \\
    \textcolor{blue}{We changed the wording to make it clearer what sets infectious disease GWAS apart: ``Infectious diseases are particularly challenging for genome-wide association studies (GWAS) because genetic effects from two organisms (pathogen and host) can influence a trait.''}
    \item L84-86. There are more than two methods for this, see also https://doi.org/10.1371/journal.pgen.1007758; https://doi.org/10.1093/bioinformatics/bty539; https://doi.org/10.1099/mgen.0.000469. Maybe remove 'two', as this will change anyway.\\
    \textcolor{blue}{We removed the ``two'' as suggested: ``More recently, analogous methods have been developed for microbial GWAS, where clonal reproduction exacerbates population structure \citep{Power2017}. Microbial GWAS-specific phylogenetic methods to account for population structure in microbial GWAS include explicitly testing for lineage-specific effects as in \citep{Earle2016} and modified association tests that account for phylogenetic relationships amongst samples as in \citep{collins_phylogenetic_nodate}.''}
    \item L110 'Of these, only pathogen effects are heritable from one transmission partner to another'. This might be nit-picking but I could see this phrasing being misunderstood by some readers. Environment will be partially shared along a transmission chain, probably. Interaction effects could also be shared if e.g. hosts are likely to have the same ancestry.\\
    \textcolor{blue}{We improved the phrasing to be more accurate: ``GWAS typically stratify samples or include covariates to correct for host genetic factors or non-genetic factors that may be correlated with a trait value. This leaves pathogen genetic factors as a remaining source of correlation, since close transmission partners may be infected with very similar pathogen strains.''}
    \item Equation 1 onwards. Using 'e' for the environmental effects is easy to get confused with more standard use of the number e = 2.718, especially when raised to a power. Consider changing this symbol.\\
    \textcolor{blue}{We changed the variable to $\epsilon$ and also changed the variable for the pathogen part of the trait to $g$ to match previous work on the POUMM (e.g. \citep{Mitov2017a-POUMM})}.
    \item Please add the link to the github code in the framework section at the end of the methods.\\
    \textcolor{blue}{We added the link in the section ``Phylogenetic spVL correction'': ``All the code used to implement the method is available at https://github.com/cevo-public/POUMM-GWAS.''}
    \item L283 'In simulations, we showed this is the case when heritability is high, selection strength is low, and trait values are not subject to strong stochastic fluctuations.' Could the authors give a biological example of this? Would antimicrobial resistance variation fit the bill?\\
    \textcolor{blue}{We anticipate antimicrobial resistance would be under strong selective pressure, but agree this could produce phylogenetically correlated trait values. We added a statement in the discussion: ``It is possible that traits under strong selection pressure would also be phylogenetically structured, for instance antimicrobial resistance, though such traits might violate the POUMM assumption that trait values vary as a random walk if they are caused by few mutations of strong affect.''}
\end{enumerate}
\end{document}