---
title: "Make manuscript figures"
output: html_notebook
---

This script is to make cleaner, consistently formatted figures for the manuscript.

Load libraries
```{r}
library(ggtree)
library(ggplot2)
library(dplyr)
library(POUMM)
library(ape)
library(cowplot)
library(treeio)

FIGDIR <- "figures"
OUTDIR <- "output"

source("scripts/R/functions/plot_functions.R")
source("scripts/R/functions/utility_functions.R")
```

Define formatting
```{r}
diverging_palette <- "Set2"
gradient_palette <- "PiYG"
shared_theme <- theme_bw()
linewidth <- 16.5  # cm
height <- 11  # cm

gv_name <- "Pathogen part of trait"
gh_name <- "Host part of trait"
e_name <- "Non-pathogen part of trait"
gvhat_name <- "Estimated pathogen part of trait"
ehat_name <- "Estimated non-pathogen part of trait"
z_minus_zbar_name <- "Scaled trait value"
```

Load data
```{r}
accuracy_simulation_results <- read.delim("output/2021-08-31_estimator_accuracy_MLE.txt")
tpr_simulation_results <- read.delim("output/2021-08-31_GWAS_TPR_MLE.txt")
tree <- ape::read.tree(file = paste(OUTDIR, "pathogen_no_outgroup.newick", sep = "/"))
traits <- read.csv(file = paste(OUTDIR, "poumm_corrected_traits.csv", sep = "/"))
tree <- treeio::full_join(tree, traits, by = "label")
POUMM_fit <- get(load(file = paste(OUTDIR, "POUMM_fit.RData", sep = "/")))
```

Model illustration
```{r}
# Simulate many OU trajectories along tree
generate_trait_trajectory <- function(v0, alpha, theta, sigma, N_steps) {
  POUMM::rTrajectoryOU(
    z0 = v0,
    t = 1,
    alpha = alpha,
    theta = theta,
    sigma = sigma,
    steps = N_steps)
}

# Generate tree
set.seed(10)
tree <- ape::rtree(n = 3)
tree$edge.length <- tree$edge.length * 6
node_times <- POUMM::nodeTimes(tree)
edge_colors = RColorBrewer::brewer.pal(n = 4, name = diverging_palette)

# Define OU process
scale_factor <- 40  # 1000
v0 <- 3.5
theta <- 4.5
alpha <- 50 / scale_factor
sigma <- 3 / sqrt(scale_factor)

# Initialize data structures
traceplot <- as.data.frame(matrix(ncol = 4, nrow = 0))
colnames(traceplot) <- c("lineage", "time", "v", "color")
v_nodes <- rep(v0, length(node_times))

# Generate OU trajectory for each edge on tree
for (j in 1:nrow(tree$edge)) {
  from = tree$edge[j, 1]
  to = tree$edge[j, 2]
  N_steps = round(tree$edge.length[j]*scale_factor) - 1
  print(paste(N_steps, "steps on branch", j))

  trajectory <- POUMM::rTrajectoryOU(
    z0 = v_nodes[from],
    t = 0.01,
    alpha = alpha,
    theta = theta,
    sigma = sigma,
    steps = N_steps)

  v_nodes[to] <- trajectory[N_steps]
  v <- c(v_nodes[from], trajectory)

  new_data <- data.frame(
    lineage = rep(j, N_steps + 1),
    time = seq(node_times[from], node_times[to], length.out = N_steps + 1),
    v = v,
    color = rep(edge_colors[j], N_steps + 1))
  traceplot <- rbind(traceplot, new_data)
}

mu_OU <- function(t) v0 * exp(-alpha * t) + theta * (1 - exp(-alpha * t))

OU_plot <- ggplot(
  data = traceplot,
  aes(x = time, y = v, group = lineage, color = I(color))) +
  geom_line(size = 1.25) +
  labs(
    x = expression('Substitutions site'^-1 * ' year'^-1),
    y = expression(italic(g)[v])) +
  shared_theme +
  stat_function(fun = mu_OU, color = "black", linetype = "dashed")

#TODO: I have no idea how the edge coloring works, this is a really shitty hardcoded soln
tree_data = data.frame(node = c(1, 2, 3, 4, 5), color = c(edge_colors[2], edge_colors[3], edge_colors[4], "blue", edge_colors[1]))

tree_plot <- ggtree::ggtree(
  tree,
  ladderize = F,
  size = 1.5) %<+% tree_data + aes(color = I(color)) +
  labs(x = expression('Substitutions site'^-1 * ' year'^-1))

e_data <- data.frame(x = c(-1, 1))

e_plot <-  ggplot(data = e_data, aes(x)) +
  stat_function(fun = dnorm,
                n = 101,
                args = list(mean = 0, sd = 0.2),
                size = 1) +
  ylab("Density") +
  shared_theme +
  labs(x = expression(italic(e)))

# Align OU and tree plots
left_side <- plot_grid(OU_plot, tree_plot, align = "v", axis = "lr", nrow = 2, rel_heights = c(2, 1), labels = c("A", "C"))
right_side <- plot_grid(NULL, e_plot, NULL, nrow = 3, rel_heights = c(1, 2, 1), labels = c("", "B", ""))

png(
  filename = paste(FIGDIR, "model_figure.png", sep = "/"),
  width = linewidth, height = height, units = "cm", res = 300)
  plot_grid(left_side, NULL, right_side, ncol = 3, rel_widths = c(1.5, 0.2, 1))
dev.off()

```

Simulation results
```{r}
# Get POUMM values for HIV spVL to annotate plot with
spvl_data <- data.frame(
        inference.method = "h.MWA.p.value",
        H2 = 3.9,
        alpha = 8.046)  # values are hand-calculated because the breaks scale is defined as seq(0.5, n.breaks - 0.5, 1) in plotting function

spvl_data$inference.method <- factor(
        spvl_data$inference.method,
        levels = c("h.plus.e.p.value", "h.MWA.p.value", "norm.z.p.value"),
        labels = c(e_name, ehat_name, z_minus_zbar_name))

# Summarize results for plotting
error.cols <- c("RMSE.z.h", "RMSE.h.MWA.h")

data.long <- do.call(
        what = tidyr::gather,
        args = c(error.cols, list(data = accuracy_simulation_results, key = "inference.method", value = "error")))

data.summary <- as.data.frame(
        data.long %>%
                dplyr::group_by(inference.method, alpha, H2, var.z, p) %>%
                dplyr::summarise(mean.error = mean(error), sd.error = sd(error), N.reps = dplyr::n()))

# Create facet variable labels for plot
data.summary$inference.method <- factor(
        data.summary$inference.method,
        levels = c("RMSE.h.MWA.h", "RMSE.z.h"),
        labels = c(ehat_name, z_minus_zbar_name))

# Plot RMSE
rmse_plot <- plotHeatmap(
        df = data.summary,
        N.sim = data.summary$N.reps[[1]],
        is.MLE = T,
        scale.limits = c(0, 1),
        legend.name = "RMSE",
        midpoint = 0.5,
        fill.hi = RColorBrewer::brewer.pal(name = 'PiYG', n = 3)[1],
        fill.lo = RColorBrewer::brewer.pal(name = 'PiYG', n = 3)[3],
        fill.mid = RColorBrewer::brewer.pal(name = 'PiYG', n = 3)[2],
        x.var = "alpha",
        y.var = "H2",
        fill.var = "mean.error",
        facet.x = "inference.method",
        facet.y = NA,
        is.oob.squish = F,
        is.real.scale = F,
        labeller = label_wrap_gen(width = 25)) +
        labs(x = expression("Selection strength (" * alpha * ")"),
             y = expression("spVL heritability (" * italic('H'^2) * ")")) +
        shared_theme +
        theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust = 1)) +
        geom_point(data = spvl_data, fill = "black")

data <- tidyr::gather(
        data = tpr_simulation_results,
        key = "inference.method",
        value = "p.value",
        h.plus.e.p.value, norm.z.p.value, h.OU.p.value, h.MWA.p.value)

# Calculate TPR for each simulation
sig.level <- correctSignificanceLevelForMultTesting(
        significance.level = 0.05,
        significance.correction = "Bonferroni",
        N.tests = 20)
data$TP <- ifelse(data$p.value < sig.level & data$is.causal.variant, T, F)
data$FP <- ifelse(data$p.value < sig.level & !(data$is.causal.variant), T, F)
data.TPR <- as.data.frame(
        data %>%
                dplyr::group_by(runID, inference.method, alpha, sigma, H2, var.z, p, delta) %>%
                dplyr::summarise(
                        TPR = sum(TP, na.rm = T)/sum(is.causal.variant),
                        FP = sum(FP), K = dplyr::n(),
                        N.noncausal.variants = sum(!(is.causal.variant))))

# Summarize replicate simulations
data.summary <- as.data.frame(
        data.TPR %>%
                dplyr::group_by(inference.method, alpha, sigma, H2, var.z, p, delta) %>%
                dplyr::summarise(
                        mean.TPR = mean(TPR),
                        sd.TPR = sd(TPR),
                        mean.FP = mean(FP),
                        sd.FP = sd(FP),
                        N.noncausal.variants = mean(N.noncausal.variants),
                        N.sim = dplyr::n()))

# Create facet variable labels for plot
data.summary$inference.method <- factor(
        data.summary$inference.method,
        levels = c("h.plus.e.p.value", "h.MWA.p.value", "norm.z.p.value"),
        labels = c(e_name, ehat_name, z_minus_zbar_name))

# Plot TPR
tpr_plot <- makeTPRPlot(
        df = data.summary %>% filter(inference.method %in% c(e_name, ehat_name, z_minus_zbar_name)),
        is.MLE = T,
        midpoint = 0.5,
        fill.lo = RColorBrewer::brewer.pal(name = 'RdBu', n = 3)[3],
        fill.hi = RColorBrewer::brewer.pal(name = 'RdBu', n = 3)[1],
        fill.mid = RColorBrewer::brewer.pal(name = 'RdBu', n = 3)[2]) +
        facet_grid(. ~ inference.method, labeller = label_wrap_gen(width = 25)) +
        labs(x = expression("Selection strength (" * alpha * ")"),
             y = expression("spVL heritability (" * 'H'^2 * ")")) +
        shared_theme +
        theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust = 1)) +
        geom_point(data = spvl_data, fill = "black")

top_row <- plot_grid(NULL, rmse_plot, rel_widths = c(1, 3.25), labels = c("", "A"))
png(filename = "figures/simulation_results.png", width = linewidth, height = height, unit = "cm", res = 300)
plot_grid(top_row, tpr_plot, align = "v", axis = "tb", nrow = 2, labels = c("", "B"))
dev.off()
```

Inferred phylogeny
```{r}
legend_width <- 15

# Plot tree with tip colors by spVL components
legend_x_pos <- 0.99
legend_y_pos <- 0.45

v_tree <- ggtree(tree, ladderize = F) +
        geom_tippoint(aes(color = v.MWA)) +
        scale_color_viridis_c(
                option = "plasma",
                direction = -1,
                limits = c(min(traits$v.MWA), max(traits$v.MWA))) +
        theme(legend.position = "bottom") +
        guides(color = guide_colorbar(title.position="top", title.hjust = 0.5)) +
        labs(color = yulab.utils::str_wrap(gvhat_name, width = legend_width))

h_tree <- ggtree(tree, ladderize = F) +
        geom_tippoint(aes(color = h.MWA)) +
        scale_color_viridis_c() +
        theme(legend.position = "bottom") +
        guides(color = guide_colorbar(title.position="top", title.hjust = 0.5)) +
        labs(color = yulab.utils::str_wrap(ehat_name, width = legend_width))

spvl_tree_same_scale <- ggtree(tree, ladderize = F) +
        geom_tippoint(aes(color = trait)) +
        scale_color_viridis_c(
                option = "plasma",
                direction = -1,
                limits = c(min(traits$v.MWA), max(traits$v.MWA))) +
        theme(legend.position = "bottom") +
        guides(color = guide_colorbar(title.position="top", title.hjust = 0.5)) +
        labs(color = yulab.utils::str_wrap("Calculated spVL", width = legend_width)) +
        scale_fill_identity(name = '', guide = 'legend', labels = c('oob'))

png(file = paste(FIGDIR, "spvl_on_tree.png", sep = "/"),
    height = height, width = linewidth,
    units = "cm",
    res = 300)
plot_grid(spvl_tree_same_scale + theme(legend.position = "none"),
          h_tree + theme(legend.position = "none"),
          v_tree + theme(legend.position = "none"),
          get_legend(spvl_tree_same_scale), get_legend(h_tree), get_legend(v_tree),
          nrow = 2, align = "v", axis = "tb",
          rel_heights = c(3, 1),
          labels = c("A", "B", "C", "", "", ""))
dev.off()
```

POUMM parameter estimates
```{r}
# Generate figures, write out estimates
plots <- plot(POUMM_fit, doZoomIn = TRUE, doPlot = F)
p <- plots$densplot +
        scale_fill_manual(
                values = RColorBrewer::brewer.pal(n = 3, name = diverging_palette),
                aesthetics = c("fill", "color"),
                labels = c("Prior", "Chain 1", "Chain 2"),
                name = element_blank()) +
        theme_bw() +
        labs(x = "Value", y = "Density") +
        scale_y_continuous(expand = c(0, 0)) +
        theme(legend.position = "bottom")
ggsave(plot = p, filename = paste(FIGDIR, "poumm_parameter_estimates.png", sep = "/"),
       width = linewidth, height = height, units = "cm")
```

Estimated trait values
```{r}

```

PCA of cohort genotypes
```{r}

```

Manhattan plot of GWAS results
```{r}

```

Q-Q plot of GWAS results
```{r}

```
