---
title: "Make manuscript figures"
output: html_notebook
---

This script is to make cleaner, consistently formatted figures for the manuscript.

Load libraries
```{r}
library(ggtree)
library(ggplot2)
library(dplyr)
library(POUMM)
library(ape)
library(cowplot)
library(treeio)
library(ggExtra)

FIGDIR <- "figures"
OUTDIR <- "output"

source("scripts/R/functions/plot_functions.R")
source("scripts/R/functions/utility_functions.R")
```

Define formatting
```{r}
diverging_palette <- "Set2"
gradient_palette <- "PiYG"
shared_theme <- theme_bw()
linewidth <- 16.5  # cm
height <- 11  # cm

gv_name <- "Pathogen part of trait"
gh_name <- "Host part of trait"
e_name <- "Non-pathogen part of trait"
gvhat_name <- "Estimated pathogen part of trait"
ehat_name <- "Estimated non-pathogen part of trait"
z_minus_zbar_name <- "Scaled trait value"
standard_gwas_name <- "GWAS with standard trait value"
corrected_gwas_name <- "GWAS with estimated non-pathogen part of trait"
genome_wide_significance <- 5E-8
```

Load data
```{r}
accuracy_simulation_results <- read.delim("output/2021-08-31_estimator_accuracy_MLE.txt")
tpr_simulation_results <- read.delim("output/2021-08-31_GWAS_TPR_MLE.txt")
tree <- ape::read.tree(file = paste(OUTDIR, "pathogen_no_outgroup.newick", sep = "/"))
traits <- read.csv(file = paste(OUTDIR, "poumm_corrected_traits.csv", sep = "/"))
tree <- treeio::full_join(tree, traits, by = "label")
POUMM_fit <- get(load(file = paste(OUTDIR, "POUMM_fit.RData", sep = "/")))
poumm_parameter_estimates <- read.csv(file = paste(OUTDIR, "poumm_parameter_estimates.csv", sep = "/"))
pc_scores <- read.csv(file = paste(OUTDIR, "shcs_vs_hapmap_pc_scores.csv", sep = "/"))
gwas_results <- read.delim(file = paste(OUTDIR, "gwas_results.pvals.effectsize.txt", sep = "/"), sep = "", header = T)
spvl_values <- read.csv(file = paste(OUTDIR, "spvl_values.csv", sep = "/"))
```

Model illustration
```{r}
# Simulate many OU trajectories along tree
generate_trait_trajectory <- function(v0, alpha, theta, sigma, N_steps) {
      POUMM::rTrajectoryOU(
            z0 = v0,
            t = 1,
            alpha = alpha,
            theta = theta,
            sigma = sigma,
            steps = N_steps)
}

# Generate tree
set.seed(10)
tree <- ape::rtree(n = 3)
tree$edge.length <- tree$edge.length * 6
node_times <- POUMM::nodeTimes(tree)
edge_colors = RColorBrewer::brewer.pal(n = 4, name = diverging_palette)

# Define OU process
scale_factor <- 40  # 1000
v0 <- 3.5
theta <- 4.5
alpha <- 50 / scale_factor
sigma <- 3 / sqrt(scale_factor)

# Initialize data structures
traceplot <- as.data.frame(matrix(ncol = 4, nrow = 0))
colnames(traceplot) <- c("lineage", "time", "v", "color")
v_nodes <- rep(v0, length(node_times))

# Generate OU trajectory for each edge on tree
for (j in 1:nrow(tree$edge)) {
      from = tree$edge[j, 1]
      to = tree$edge[j, 2]
      N_steps = round(tree$edge.length[j]*scale_factor) - 1
      print(paste(N_steps, "steps on branch", j))

      trajectory <- POUMM::rTrajectoryOU(
            z0 = v_nodes[from],
            t = 0.01,
            alpha = alpha,
            theta = theta,
            sigma = sigma,
            steps = N_steps)

      v_nodes[to] <- trajectory[N_steps]
      v <- c(v_nodes[from], trajectory)

      new_data <- data.frame(
            lineage = rep(j, N_steps + 1),
            time = seq(node_times[from], node_times[to], length.out = N_steps + 1),
            v = v,
            color = rep(edge_colors[j], N_steps + 1))
      traceplot <- rbind(traceplot, new_data)
}

mu_OU <- function(t) v0 * exp(-alpha * t) + theta * (1 - exp(-alpha * t))

OU_plot <- ggplot(
      data = traceplot,
      aes(x = time, y = v, group = lineage, color = I(color))) +
      geom_line(size = 1.25) +
      labs(
        x = element_blank(),
        y = expression(italic(g)[v])) +
      shared_theme +
      stat_function(fun = mu_OU, color = "black", linetype = "dashed")

#TODO: I have no idea how the edge coloring works, this is a really shitty hardcoded soln
tree_data = data.frame(node = c(1, 2, 3, 4, 5), color = c(edge_colors[2], edge_colors[3], edge_colors[4], "blue", edge_colors[1]))

tree_plot <- ggtree::ggtree(
      tree,
      ladderize = F,
      size = 1.5) %<+% tree_data + aes(color = I(color)) +
      labs(x = expression('Substitutions site'^-1 * ' year'^-1))

e_data <- data.frame(x = c(-1, 1))

e_plot <-  ggplot(data = e_data, aes(x)) +
      stat_function(fun = dnorm,
                    n = 101,
                    args = list(mean = 0, sd = 0.2),
                    size = 1) +
      ylab("Density") +
      shared_theme +
      labs(x = expression(italic(e)))

# Align OU and tree plots
left_side <- plot_grid(OU_plot, tree_plot, align = "v", axis = "lr", nrow = 2, rel_heights = c(3, 1), labels = c("A", "C"))
right_side <- plot_grid(NULL, e_plot, NULL, nrow = 3, rel_heights = c(1, 2, 1), labels = c("", "B", ""))

png(
      filename = paste(FIGDIR, "model_figure.png", sep = "/"),
      width = linewidth * 2/3, height = height * 2/3, units = "cm", res = 300)
      plot_grid(left_side, NULL, right_side, ncol = 3, rel_widths = c(1.5, 0.1, 1))
dev.off()

```

Simulation results
```{r}
# Get POUMM values for HIV spVL to annotate plot with
H2_estimate <- poumm_parameter_estimates[poumm_parameter_estimates$stat == "H2tMean", "PostMean"]
alpha_estimate <- poumm_parameter_estimates[poumm_parameter_estimates$stat == "alpha", "PostMean"]

# Breaks scale is defined as seq(0.5, n.breaks - 0.5, 1) in plotting function, need to translate real values to breaks scale
get_xval_for_sim_plots <- function(real_vals, estimate) {
      alpha_vals <-
      base_val <- rev(real_vals[real_vals < estimate])[1]
      next_val <- real_vals[real_vals > estimate][1]
      base_xval <- which(real_vals == base_val)
      next_xval <- which(real_vals == next_val)
      xval <- base_xval - 0.5 + (next_xval - base_xval) * (estimate - base_val) / (next_val - base_val)
      return(xval)
}

spvl_data <- data.frame(
        inference.method = "h.MWA.p.value",
        H2 = get_xval_for_sim_plots(real_vals = c(0.15, 0.25, 0.35, 0.45, 0.55, 0.65, 0.75), estimate = H2_estimate),
        alpha = get_xval_for_sim_plots(real_vals = c(0.1, 0.5, 1, 5, 10, 20, 30, 40, 50, 60), estimate = alpha_estimate))

spvl_data$inference.method <- factor(
        spvl_data$inference.method,
        levels = c("h.plus.e.p.value", "h.MWA.p.value", "norm.z.p.value"),
        labels = c(e_name, ehat_name, z_minus_zbar_name))

# Summarize results for plotting
error.cols <- c("RMSE.z.h", "RMSE.h.MWA.h")

data.long <- do.call(
        what = tidyr::gather,
        args = c(error.cols, list(data = accuracy_simulation_results, key = "inference.method", value = "error")))

data.summary <- as.data.frame(
        data.long %>%
                dplyr::group_by(inference.method, alpha, H2, var.z, p) %>%
                dplyr::summarise(mean.error = mean(error), sd.error = sd(error), N.reps = dplyr::n()))

# Create facet variable labels for plot
data.summary$inference.method <- factor(
        data.summary$inference.method,
        levels = c("RMSE.h.MWA.h", "RMSE.z.h"),
        labels = c(ehat_name, z_minus_zbar_name))

# Plot RMSE
rmse_plot <- plotHeatmap(
        df = data.summary,
        N.sim = data.summary$N.reps[[1]],
        is.MLE = T,
        scale.limits = c(0, 1),
        legend.name = "RMSE",
        midpoint = 0.5,
        fill.hi = RColorBrewer::brewer.pal(name = 'PiYG', n = 3)[1],
        fill.lo = RColorBrewer::brewer.pal(name = 'PiYG', n = 3)[3],
        fill.mid = RColorBrewer::brewer.pal(name = 'PiYG', n = 3)[2],
        x.var = "alpha",
        y.var = "H2",
        fill.var = "mean.error",
        facet.x = "inference.method",
        facet.y = NA,
        is.oob.squish = F,
        is.real.scale = F,
        labeller = label_wrap_gen(width = 25)) +
        labs(x = expression("Selection strength (" * alpha * ")"),
             y = expression("spVL heritability (" * italic('H'^2) * ")")) +
        shared_theme +
        theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust = 1)) +
        geom_point(data = spvl_data, fill = "black")

data <- tidyr::gather(
        data = tpr_simulation_results,
        key = "inference.method",
        value = "p.value",
        h.plus.e.p.value, norm.z.p.value, h.OU.p.value, h.MWA.p.value)

# Calculate TPR for each simulation
sig.level <- correctSignificanceLevelForMultTesting(
        significance.level = 0.05,
        significance.correction = "Bonferroni",
        N.tests = 20)
data$TP <- ifelse(data$p.value < sig.level & data$is.causal.variant, T, F)
data$FP <- ifelse(data$p.value < sig.level & !(data$is.causal.variant), T, F)
data.TPR <- as.data.frame(
        data %>%
                dplyr::group_by(runID, inference.method, alpha, sigma, H2, var.z, p, delta) %>%
                dplyr::summarise(
                        TPR = sum(TP, na.rm = T)/sum(is.causal.variant),
                        FP = sum(FP), K = dplyr::n(),
                        N.noncausal.variants = sum(!(is.causal.variant))))

# Summarize replicate simulations
data.summary <- as.data.frame(
        data.TPR %>%
                dplyr::group_by(inference.method, alpha, sigma, H2, var.z, p, delta) %>%
                dplyr::summarise(
                        mean.TPR = mean(TPR),
                        sd.TPR = sd(TPR),
                        mean.FP = mean(FP),
                        sd.FP = sd(FP),
                        N.noncausal.variants = mean(N.noncausal.variants),
                        N.sim = dplyr::n()))

# Create facet variable labels for plot
data.summary$inference.method <- factor(
        data.summary$inference.method,
        levels = c("h.plus.e.p.value", "h.MWA.p.value", "norm.z.p.value"),
        labels = c(e_name, ehat_name, z_minus_zbar_name))

# Plot TPR
tpr_plot <- makeTPRPlot(
        df = data.summary %>% filter(inference.method %in% c(e_name, ehat_name, z_minus_zbar_name)),
        is.MLE = T,
        midpoint = 0.5,
        fill.lo = RColorBrewer::brewer.pal(name = 'RdBu', n = 3)[3],
        fill.hi = RColorBrewer::brewer.pal(name = 'RdBu', n = 3)[1],
        fill.mid = RColorBrewer::brewer.pal(name = 'RdBu', n = 3)[2]) +
        facet_grid(. ~ inference.method, labeller = label_wrap_gen(width = 25)) +
        labs(x = expression("Selection strength (" * alpha * ")"),
             y = expression("spVL heritability (" * italic('H'^2) * ")")) +
        shared_theme +
        theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust = 1)) +
        geom_point(data = spvl_data, fill = "black")

top_row <- plot_grid(NULL, rmse_plot, rel_widths = c(1, 3.25), labels = c("", "A"))
png(filename = "figures/simulation_results.png", width = linewidth, height = height, unit = "cm", res = 300)
plot_grid(top_row, tpr_plot, align = "v", axis = "tb", nrow = 2, labels = c("", "B"))
dev.off()
```

Inferred phylogeny
```{r}
legend_width <- 15

# Plot tree with tip colors by spVL components
legend_x_pos <- 0.99
legend_y_pos <- 0.45

v_tree <- ggtree(tree, ladderize = F) +
        geom_tippoint(aes(color = v.MWA)) +
        scale_color_viridis_c(
                option = "plasma",
                direction = -1,
                limits = c(min(traits$v.MWA), max(traits$v.MWA))) +
        theme(legend.position = "bottom") +
        guides(color = guide_colorbar(title.position="top", title.hjust = 0.5)) +
        labs(color = yulab.utils::str_wrap(gvhat_name, width = legend_width))

h_tree <- ggtree(tree, ladderize = F) +
        geom_tippoint(aes(color = h.MWA)) +
        scale_color_viridis_c() +
        theme(legend.position = "bottom") +
        guides(color = guide_colorbar(title.position="top", title.hjust = 0.5)) +
        labs(color = yulab.utils::str_wrap(ehat_name, width = legend_width))

spvl_tree_same_scale <- ggtree(tree, ladderize = F) +
        geom_tippoint(aes(color = trait)) +
        scale_color_viridis_c(
                option = "plasma",
                direction = -1,
                limits = c(min(traits$v.MWA), max(traits$v.MWA))) +
        theme(legend.position = "bottom") +
        guides(color = guide_colorbar(title.position="top", title.hjust = 0.5)) +
        labs(color = yulab.utils::str_wrap("Calculated spVL", width = legend_width)) +
        scale_fill_identity(name = '', guide = 'legend', labels = c('oob'))

png(file = paste(FIGDIR, "spvl_on_tree.png", sep = "/"),
    height = height, width = linewidth,
    units = "cm",
    res = 300)
plot_grid(spvl_tree_same_scale + theme(legend.position = "none"),
          h_tree + theme(legend.position = "none"),
          v_tree + theme(legend.position = "none"),
          get_legend(spvl_tree_same_scale), get_legend(h_tree), get_legend(v_tree),
          nrow = 2, align = "v", axis = "tb",
          rel_heights = c(3, 1),
          labels = c("A", "B", "C", "", "", ""))
dev.off()
```

POUMM parameter estimates
```{r}
# Generate figures, write out estimates
plots <- plot(POUMM_fit, doZoomIn = TRUE, doPlot = F)
p <- plots$densplot +
        scale_fill_manual(
                values = RColorBrewer::brewer.pal(n = 3, name = diverging_palette),
                aesthetics = c("fill", "color"),
                labels = c("Prior", "Chain 1", "Chain 2"),
                name = element_blank()) +
        theme_bw() +
        labs(x = "Value", y = "Density") +
        scale_y_continuous(expand = c(0, 0)) +
        theme(legend.position = "bottom")
ggsave(plot = p, filename = paste(FIGDIR, "poumm_parameter_estimates.png", sep = "/"),
       width = linewidth, height = height, units = "cm")
```

Estimated trait values
TODO: think about some way of depicting phylogenetic relationships here too?
```{r}
p <- ggplot(data = spvl_values,
            aes(x = spvl_lenient_filter_first_measurement,
                y = spvl_lenient_filter_mean_measurement,
                color = spvl_strict_filter_mean_measurement)) +
        scale_color_gradient(low = "green", high = "red", name = "Mean measurement\nwith stricter filtering") +
        coord_equal() +
        geom_point() +
        labs(x = "First measurement", y = "Mean measurement") +
        shared_theme

ggsave(plot = p, filename = paste(FIGDIR, "spvl_calculation_comparison.png", sep = "/"),
       width = linewidth * 2/3, height = height * 2/3, units = "cm")

p1 <- ggplot(data = traits, aes(x = trait, y = h.MWA)) +
      geom_point() +
      shared_theme +
      labs(x = "Calculated spVL trait", y = ehat_name)
p2 <- ggExtra::ggMarginal(p1, type="histogram")
ggsave(plot = p2, filename = paste(FIGDIR, "trait_values.png", sep = "/"),
      width = linewidth * 2/3, height = height * 2/3, units = "cm")
```

PCA of cohort genotypes
```{r}
# Plot combined SHCS and HapMap samples along the first few PCs
pop_colors <- RColorBrewer::brewer.pal(n = 11, name = "Set3")
names(pop_colors) <- unique(pc_scores$Population)
swap_col <- pop_colors[9]
pop_colors[9] <- pop_colors[4]
pop_colors[4] <- swap_col

p1 <- ggplot(
        data = pc_scores %>% pivot_longer(cols = c(PC2, PC3), names_to = "PC_number", values_to = "PC_value"),
        aes(x = PC1, y = PC_value, fill = Population, color = is_european)) +
        geom_point(pch=21, size=3) +
        shared_theme +
        scale_fill_manual(values = pop_colors) +
        scale_color_manual(values = c("transparent", "black")) +
        labs(y = element_blank()) +
        guides(color = "none") +
        facet_grid(PC_number ~ .)

ggsave(plot = p1, filename = paste(FIGDIR, "host_genotype_pca.png", sep = "/"),
       width = linewidth, height = height, units = "cm")
```

Q-Q plot of GWAS results
```{r}
qq_data_standard <- data.frame(
        observed_log10p = -log10(sort(gwas_results$P_standard, decreasing = F)),
        expected_log10p = -log10(ppoints(nrow(gwas_results))),
        gwas_type = standard_gwas_name)
qq_data_phylo <- data.frame(
        observed_log10p = -log10(sort(gwas_results$P_corrected, decreasing = F)),
        expected_log10p = -log10(ppoints(nrow(gwas_results))),
        gwas_type = corrected_gwas_name)
qq_results <- rbind(qq_data_standard, qq_data_phylo)

p <- ggplot(data = qq_results, aes(x = expected_log10p, y = observed_log10p)) +
        geom_point() +
        geom_abline(slope = 1, intercept = 0, linetype = "dashed",
                    color = RColorBrewer::brewer.pal(n = 3, name = diverging_palette)[[1]]) +
        facet_wrap(. ~ gwas_type) +
        labs(x = expression("expected log"[10]*"(p)"), y = expression("observed log"[10]*"(p)")) +
        shared_theme
ggsave(plot = p, filename = paste(FIGDIR, "qq_plots.png", sep = "/"),
       width = linewidth, height = height * 2/3, units = "cm")
```

Manhattan plot of GWAS results
```{r}
pt_size <- 1

# Add genome position to data
chr_lengths <- gwas_results %>%
        group_by(CHROM) %>%
        summarise(chr_len = max(POS))
chr_starts <- chr_lengths %>%
        mutate(CHR_start = cumsum(as.numeric(chr_len)) - chr_len) %>%
        select(-chr_len)
gwas_results_annotated <- left_join(gwas_results, chr_starts, by = "CHROM")
gwas_results_annotated <- gwas_results_annotated %>%
        arrange(CHROM, POS) %>%
        mutate(BP_position = CHR_start + POS)

# This is maybe not the best way, but makes sure tick marks and labels are at the center of each chromosome
max_BP_position <- max(gwas_results_annotated[gwas_results_annotated$CHROM == 22, "BP_position"])
x_axis_ticks <- chr_starts
x_axis_ticks <- rbind(
        x_axis_ticks,
        data.frame(
                CHROM = c(23, 24),
                CHR_start = c(max_BP_position, max_BP_position + 1000)))
x_axis_ticks$midpoint <- x_axis_ticks$CHR_start + (lead(x_axis_ticks$CHR_start) - x_axis_ticks$CHR_start)/2

# Format data for plotting
results_long <- tidyr::pivot_longer(
        data = gwas_results_annotated,
        cols = c("P_standard", "P_corrected"),
        names_to = "gwas_type",
        names_prefix = "P_",
        values_to = "P")

# Fix facet labels for plotting
results_long$gwas_type <- factor(
        results_long$gwas_type,
        levels = c("standard", "corrected"),
        labels = c(standard_gwas_name, corrected_gwas_name))

# Make comparative GWAS overview plot
overview_comparison <- ggplot(
        data = results_long %>% sample_n(size = 10000),  ###
        aes(x = BP_position, y = -log10(P))) +
        geom_point(aes(color = CHROM %% 2 == 0), size = pt_size) +
        scale_color_manual(values = c("grey", "dark grey")) +
        scale_x_continuous(
                breaks = x_axis_ticks$midpoint,
                labels = x_axis_ticks$CHROM,
                expand = c(0.01, 0.01),  # neccessary to fit chrom 23 label on
                guide = guide_axis(n.dodge = 2)) +  # makes the x-axis labels alternate hi/lo
        scale_y_continuous(limits = c(0, 16), expand = c(0, 0)) +
        labs(x = "Chromosome", y = expression('-Log'[10]*'(p)')) +
        geom_hline(
                yintercept = -log10(genome_wide_significance),
                linetype = "dashed") +
        shared_theme +
        theme(
                panel.grid.major.x = element_blank(),
                panel.grid.minor.x = element_blank(),
                legend.position = "none",
                axis.text.x = element_text(size = 5)
        ) +
        facet_grid(. ~ gwas_type)

# Make plot showing p-value changes in the CCR5 and MHC regions
regional_results<- gwas_results_annotated %>% filter(
        (CHROM == 3 & POS >= 45.5E6 & POS <= 47E6) | (CHROM == 6 & POS >= 29.5E6 & POS <= 33.5E6))

# Fix facet labels for plotting
regional_results$CHROM <- factor(
        regional_results$CHROM,
        levels = c("3", "6"),
        labels = c("italic(CCR5)*' region'", "'MHC region'"))

p_vals_regional <- ggplot(
        data = regional_results  %>% sample_n(size = 100),  ###
        aes(x = POS/1E6, y = -log10(P_corrected))) +
        geom_point(aes(color = -neg_log10_P_standard_minus_corrected), size = pt_size) +
        scale_color_gradient2(
                name = expression(Delta*'[-Log'[10]*'(p)]'),
                low = "#B2182B",
                mid = "#FFFFBF",
                high = "#2166AC",
                midpoint = 0) +
        scale_y_continuous(limits = c(0, 16), expand = c(0, 0)) +
        labs(
                x = "Base position (Mb)",
                y = expression('-Log'[10]*'(p)')) +
        geom_hline(
                yintercept = -log10(genome_wide_significance),
                linetype = "dashed") +
        shared_theme +
        theme(
                panel.grid.major.x = element_blank(),
                panel.grid.minor.x = element_blank()) +
        facet_grid(. ~ CHROM, labeller = label_parsed, scales = "free")

# Assemble the plots as subplots
aligned_plots <- align_plots(overview_comparison, p_vals_regional, align = "h", axis = "tb")

png(
        filename = paste(FIGDIR, "gwas_results.png", sep = "/"),
        width = linewidth, height = height, unit = "cm", res = 300)
plot_grid(aligned_plots[[1]], aligned_plots[[2]], nrow = 2, labels = c("A", "B"))
dev.off()
```

